# Introduction

<!-- Chapter 1 -->
## Antimicrobial resistance
### The problem of antimicrobial resistance
Antimicrobial resistance (AMR) is a global threat to human and animal health alike, and the World Health Organization (WHO) urged for a "One Health" approach when publishing the Global Action Plan on AMR in 2015 [@WHO2015a]. Multiple definitions on antimicrobial resistance exist based on different criteria and settings. In a clinical setting, antimicrobial resistance is defined as the ability of micro-organisms to to survive antimicrobial treatment that would normally kill them [@WHO2018]. In a laboratory setting, the micro-organism is regarded as resistant if it grows in the presence of antimicrobials at a higher concentration than phylogenetically related strains [@Guardabassi2006]. Quinolone resistance is especially important to tackle because quinolones and fluoroquinolones are on the WHO list of critically important antimicrobials for human medicine [@WHO2019]. This means that quinolones are frequently used to treat human infections, and also in some cases the only antimicrobial that can treat the infection in question [@WHO2019]. Resistance to these compounds would therefore pose a great health risk. Systematic misuse of antimicrobials have been attributed to the increased emergence of AMR [@WHO2015a]. Already in 1945, Alexander Fleming warned about the misuse of antimicrobials, stating that exposing bacteria to sub-lethal concentrations in the lab rendered them resistant, and that this also may happen in the body [@Fleming1945]. However, other factors, such as environmental stress factors, is also linked to the development of AMR in the absence of antimicrobial selective pressure [@McMahon2007;@Poole2012].

> *"It is not difficult to make microbes resistant to penicillin in the laboratory by exposing them to concentrations not sufficient to kill them, and the same thing occasionally happened in the body"*
>
> `r tufte::quote_footer("--- Sir Alexander Fleming, Nobel lecture, 1945")`

### AMR transmission
Transmission of AMR happens either by vertical inheritance or through horizontal transfer of genetic material from one cell to another. Horizontal transfer is mediated by either transformation, transduction or conjugation. In vertical inheritance, resistance traits are passed on to the offspring by cell division.




<!-- Chapter 2 -->
## Quinolones and Quinolone resistance mechanisms
### Quinolones
Quinolones were introduced to the market in the late 1960s [@Emmerson2003]. The first quinolone, Nalidixic acid, was discovered as a by-product of chloroquine synthesis [@Lesher1962], and was clinically in use in 1967. A few decades later, the synthesis of novel quinolones increased the effectiveness of nalidixic acid towards gram-positive bacteria, and reduced its toxicity by altering its chemical structure. This structural change involved, among others, the addition of a fluoro-group to the molecule, which also increased its potency towards _Enterobacteriaceae_ [@Emmerson2003]. This also lead to a change in nomenclature for these compounds, as the molecules that harboured this structural change now were called fluoroquinolones.

The main target for quinolones and fluoroquinolones (hereafter called quinolones) in _Escherichia coli_ (_E. coli_), is the type II topoisomerases DNA gyrase (also known as topoisomerase II) and topoisomerase IV [@Drlica1997]. The type II topoisomerases regulate DNA topology by cutting DNA strands, passing another DNA strand through the break, and then seal the break [@Hooper1998]. Both DNA gyrase and Topoisomerase IV is comprised of two subunits, A and B. These are encoded by the genes _gyrA_ and _gyrB_ for DNA gyrase, and _parC_ and _parE_ for topoisomerase IV. The type II topoisomerases are the main target for quinolones in _E. coli_, especially the DNA gyrase. Quinolones bind to these enzymes through a water-metal ion bridge (Figure \@ref(fig:watermetalion)), connecting the quinolone molecule to two specific amino acids; serine (S) 83 and aspartic acid (D) 87 in DNA gyrase, and S 80 and glutamic acid (E) 84 in topoisomerase IV [@Aldred2014]. This binding inhibits the ligation function of the enzyme, and fragments the bacterial chromosome, leading to cell death.

![Overview of the water-metal ion bridge that mediates the quinolone-topoisomerase IV interaction, reprinted from the American Chemical Society [@Aldred2014]. \label{fig:watermetalion}](images/watermetalion.jpg)


### Quinolone resistance mechanisms in _E. coli_
Multiple mechanisms that infer resistance towards quinolones have been discovered in _E. coli_, and they are generally divided into four categories: chromosomal mutations, reduced membrane permeability, efflux pump overexpression, and plasmid mediated quinolone resistance (PMQR)[@Correia2017;@Aldred2014a], see Figure \@ref(fig:mechanisms). This is the traditional way of grouping quinolone resistance mechanisms. However, the underlying mechanism is mutations in the chromosome, that either lead to structural changes of the encoded proteins or changes in expression through various pathways. The four categories are therefore described as two different categories below: chromosomal mutations and PMQR.

![Quinolone resistance mechanisms, grouped into the traditional categories. Reprinted from Microbiology Society [@Correia2017]. \label{fig:mechanisms}](images/mechanisms.jpg){out.size=120%}



#### Chromosomal mutations
Perhaps the most well-studied resistance mechanism towards quinolones is mutations in the genes that code for the target molecules, _gyrA_, _gyrB_, _parC_, and _parE_. Missense mutations in these genes that lead to a substitution of critical amino acids needed for the binding of quinolones to the target molecules may lead to an increased tolerance towards these compounds. In _E. coli_, substitutions in a stretch of amino acids known as the quinolone resistance determining region (QRDR) in these four genes is highly correlated with quinolone resistance. In GyrA, this region is between amino acid (AA) 67 to 106 [@Yoshida1991], in GyrB between AA 333 and 481, ParC between AA 51 and 170, and ParE between AA 366 and 523. Specific substitutions within these areas are often identified in QREC, e.g. the S 83 and D 87 positions in GyrA [@Barnard2001], or the S 80 position in ParC.

Porin channels in gram negative bacterial membranes are essential for the entry of quinolones into the cell [@Correia2017]. Specific chromosomal regulons are involved in the expression of these membrane proteins, exemplified by the _mar_, _soxRS_ and _rob_ regulons [@Correia2017]. Mutations in these genes may decrease the expression of the Omp-type porins [@Oethinger1998;@Cohen1989;@Koutsolioutsou2005;@Amabile-Cuevas1991;@White1997].

In _E. coli_, the AcrAB-TolC efflux pump has been identified as a major facilitator of quinolone efflux [@Sato2013;@Weston2017]. Mutations in specific regulatory genes such as _marR_ may activate _acrAB_ and _tolC_, thus increasing their expression [@Alekshun1999a]. This will in turn reduce the drug concentration in the cell. Certain mutations in RNA polymerase subunit B (_rpoB_) have been shown to increase the expression of MdtK, which is a multidrug efflux transporter, resulting in an increase in levofloxacin MIC values [@Pietsch2017].

Mutator genes code for proteins that are involved with DNA repair mechanisms, such as methyl-directed mismatch repair (MMR; MutH, MutL, MutS, UvrD), proof-reading (MutD), adenine methylation (Dam), and oxidative damage prevention (MutT, MutM, MutY) [@Modrich1989;@Miller1996;@Chopra2003]. Favourable mutations, such as those conferring antimicrobial resistance, are relatively rare. However, mutations that impair the function of these mutator proteins may lead to a mutator phenotype [@Miller1996;@Wiegand2008], i.e. an elevated mutation rate, which may accelerate adaptation and increase the chance for a favorable mutation [@Taddei1997]. 

#### Plasmid mediated quinolone resistance
Some proteins that infer resistance to quinolones are transcribed from plasmids, such as the quinolone resistance proteins (Qnr), the efflux pumps OqxAB and QepA, and the acetyltransferase Aac(6')-_Ib-cr_.

Qnr proteins are pentapeptide proteins that are capable of protecting DNA gyrase and/or topoisomerase IV from quinolone action. Qnr proteins bind to topoisomerases and prevents the quinolone from stabilizing the lethal gyrase-DNA-quinolone cleavage complex [@Tran2005;@Tran2005a;@Robicsek2006], effectively reducing the possible targets for the quinolones. Multiple subtypes of Qnr proteins have been identified, namely QnrA, QnrB, QnrC, QnrD, QnrS, and QnrVC [@Strahilevitz2009]. Other PMQR genes have been identified, such as _mfpA_, _mfpA~MT~_, and _mcbG_. Similar to the _qnr_ genes, these genes code for pentapeptide proteins thought to have similar effect as Qnr proteins [@Strahilevitz2009;@Montero2001;@Hegde2005;@Garrido1988].

The efflux pump OqxAB has been shown to increase resistance towards chloramphenicol and quinolones, and is dependent on the host TolC outer membrane protein in _E. coli_ [@Hansen2004;@Hansen2007]. Similarly, QepA expression has been found to increase resistance towards ciprofloxacin, norfloxacin and enrofloxacin due to efflux [@Yamane2007].

The acetyltransferase Aac(6')-_Ib-cr_ infers resistance towards quinolones by altering the quinolone molecule itself. The enzyme acetylates the amino nitrogen on the piperazinyl substituent of ciprofloxacin and norfloxacin [@Robicsek2006a], which reduces the activity of the quinolone [@Robicsek2006a].




<!-- Chapter 3 -->
## _Escherichia coli_
### General bacteriology and ecology
_E. coli_ is the most studied micro-organism on earth [@Vila2016], and is often used as a model organism in various microbiology studies and as an indicator organism in surveillance. _E. coli_ is a Gram-negative, rod-shaped, facultative anaerobic bacterium often found in the gastrointestinal tract of warm-blooded animals and humans. There, it may be present as a commensal or as a pathogen.

### Clonal or not clonal?
Bacteria may evolve either by vertical inheritance, i.e. clonally, or by horizontal transfer [@Daubin2016]. A cell's genome need to be stable enough for it to be able to pass on accurate information to its offspring. However, the genome also need to be dynamic to allow genetic changes in response to selective pressures from the environment [@Levene2002]. Therefore, a successful genome has to maintain a balance between stability and flexibility in order to survive and evolve in an environment. Diversity within bacterial populations is generated by either mutation or by recombination [@Yasbin2002]. Mutation happens when nucleotides are either removed, inserted, or exchanged with other nucleotides. Recombination is the exchange of genetic material imported to the cell through either transformation, transduction or conjugation, or by recombining parts of the cell's own genome. Mutation and recombination both play important roles in the evolution of _E. coli_, but the impact of recombination is highly debated. In general, _E. coli_ is a genetically diverse species, with over 7000 sequence types [@Wirth2006] and multiple phylogenetic groups defined within the species, namely A, B1, B2, and D (sometimes paraphyletic) in addition to minor subgroups C and E. These phylogroups represent lineages within the _E. coli_ species (Figure  \@ref(fig:popstructure)). Despite the apparent diversity, the extent of recombination in _E. coli_ populations has been hypothesized to play a small role [@Selander1987;@Ochman1984a;@Selander1980] and seem to only affect short regions of the genome, which suggests that even with recombination the population structure appears clonal [@Touchon2009]. Evidence for clonal evolution was also found in studies encompassing electrophoretic movement of enzymes and sequencing analysis [@Milkman1973;@Ochman1984a;@Selander1987;@Selander1980]. 

![Phylogenetic groups of E. coli, based on maximum likelihood analysis of a concatenated alignment of 2173 genes, reprinted from Infection, Genetics and Evolution [@Chaudhuri2012]. \label{fig:popstructure}](images/pop_structure_chaudhuri.png)


\newpage

<!-- Chapter 4 -->
## Norwegian husbandry
Norwegian husbandry is characterized by low animal density, a healthy animal population, and an overall good welfare situation [@Alvseike2018]. Animal density is regulated by specific consession limits that are stricter than EU regulations [@Lovdata2004a]. Furthermore, the overall good health of the animal population reduces the need for antimicrobial use. Prophylactic use of antimicrobial compounds in Norwegian livestock is prohibited, and antimicrobials are only used to treat diseased animals. The use of quinolones to treat livestock in Norway is low [@NORM/NORM-VET2017;@ESVAC2018], see Figure \@ref(fig:quinUsage), and negligible amounts of quinolones have been sold in Norway from 2010 to 2016 [@ESVAC2018]. In this section, the broiler and pig production chains are briefly described.

![Overall sales, in kg active substance, of quinolones and macrolides for therapeutic use in terrestrial livestock animals in Norway in 1993 - 2017, reprinted from NORM/NORM-VET 2017 [@NORM/NORM-VET2017]. \label{fig:quinUsage}](images/quin_usage.png)

### Broiler production
The Norwegian broiler production is a pyramidal structure with pure-bred animals at the top, breeding animals in the middle, and broilers at the bottom [@Mo2016a]. Eggs are imported from grandparent animals in Sweden which originate mostly from Scotland or Germany. The eggs are hatched in Norwegian hatcheries, and become parent animals. These are reared for around 18 weeks, and are subsequently used for breeding of broiler chickens until around 60 weeks old. The broiler chickens are hatched in hatcheries, and distributed to broiler farms all over the country. Only a few hatcheries exist in Norway. Most of the broiler chickens are reared for 28 - 32 days, depending on weight, then slaughtered [@Nortura2015]. The broiler farms have high biosecurity, i.e. measures are taken to prevent exposure to harmful agents.

### Swine production
Swine production in Norway also has a pyramidal structure with unidirectional flow of animals, similar to the broiler production [@Kristoffersen2016]. However, the import of live pigs from other countries to the commercial pig population is negligible [@Animalia2018]. The genetic nucleus herds, where the pure-bred grand parent animals are produced, are at the top of the pyramid. Hybrid parent animals are produced in multiplier herds, either by recruiting pure-bred animals from nucleus herds or having a self-recruitment strategy. Gilts from the multiplier herds are distributed to piglet producing herds (either farrow to finish or farrow to grower herds). Additionally, sow pool systems exists in Norway. In the central units of the sow pools, sows are inseminated and kept for the majority of the gestation period up until approx. three weeks before farrowing. At this time, the sows are moved to satellite units of the sow pool system where they are kept before being moved back to the central unit after the piglets have been weaned. The piglet producing herds have a mean weaning age of 33 days, followed by a growing stage until the grower pig is approx. 30 kg and 10-11 weeks of age. The grower pig is then transferred to a specialized fattening pig farm or fattening unit at the same farm.


\newpage

<!-- Chapter 5 -->
## Monitoring AMR and QREC epidemiology
### Antimicrobial resistance surveillance

Harmonized monitoring of AMR in commensal _E. coli_ from food and food-producing animals has since 2014 been obligatory for all EU/EEA member states, as implemented by the European Commission (SANCO/11591/2012). The commission describes the sampling frame and analysis of isolates. Commensal _E. coli_ is usually isolated from caecal samples, and antimicrobial susceptibility testing (AST) is carried out using broth microdilution. With this method, the minimum inhibitory concentration (MIC) value is determined for multiple antimicrobials. A MIC value is defined as the lowest concentration of the antimicrobial needed to inhibit growth of a micro-organism [@Andrews2001]. Based on the MIC value, an isolate may be categorized as either sensitive or resistant, based on breakpoints defined by the European Comittee on Antimicrobial Susceptibility Testing (EUCAST). In a clinical setting, the clinical breakpoint is used to determine if the treatment will be successful [@Aarestrup2007a]. However, for monitoring purposes, the epidemiological breakpoint (ECOFF) is frequently used. The ECOFF is the highest MIC value for the susceptible (wild type) population [@Aarestrup2007a;EFSA2008], see Figure \@ref(fig:ecoff). It is important to distinguish between these two breakpoints, as isolates categorized as resistant with the ECOFF value may still respond to treatment in a clinical setting. Using the ECOFF enables detection of resistant isolates before the potential development of clinical resistance, and therefore works as a precautionary measure.

![Distribution of MIC-values for ciprofloxacin in E. coli. Data: www.eucast.org. Illustration: H. Kaspersen. \label{fig:ecoff}](images/micdist.png)

Guidelines from EFSA and ECDC specify the nomenclature of the occurrence of AMR in a population [@EFSA2016], which is often reported as a percentage. These percentages are usually converted to relative terms like "low" and "high". The percentage values that correspond to these relative terms are "rare" (< 0.1%), "very low" (0.1 - 1%), "low" (> 1 - 10%), "moderate" (> 10 - 20%), "high" (> 20 - 50%), "very high" (> 50 - 70%), and "extremely high" (> 70%).

### The Norwegian monitoring programme {#norwmonitor}
As part of the Norwegian government's strategy plan against antimicrobial resistance, the Norwegian monitoring programme for antimicrobial resistance in feed, food and animals (NORM-VET) was implemented in 2000. The programme has since 2014 followed the recommendations for surveillance as given by the EU (2013/652/EU) in parallel with national surveillance needs. A variety of animal species have been sampled in NORM-VET, but all animals are not sampled each year due to the sampling regime. The sample material is commonly faecal or caecal matter, but meat and boot swab samples are also included some years. In broiler and turkey flocks, 10 samples are collected from each flock and pooled before culturing. For pigs and cattle, one individual is sampled per herd. Wild animals are sampled individually. 

Standardized culturing and isolation methods are used to identify _E. coli_ from the samples collected. These isolates are used as an indicator for the presence of resistance in the given animal population on a national level. Quinolone resistance in indicator _E. coli_ has been monitored in NORM-VET since 2000. First, samples are plated onto lactose-bromothymol blue agar (pre-2013) or MacConkey agar (2013 until today). Then, a random _E. coli_ colony is subjected to susceptibility testing. In 2014, a selective method was introduced in addition to the indicator _E. coli_. Here, the samples were plated onto MacConkey agar with 0.06 mg/L ciprofloxacin. In both methods, one random colony of _E. coli_ was selected and subjected to susceptibility testing.

### Epidemiology of quinolone resistant _E. coli_
#### In Europe
Occurrence of QREC in Europe seem to follow a north - south trend, where the countries in the north have the lowest occurrence, while the southern countries have the highest, see Figure \@ref(fig:ecdcData). However, the occurrence is markedly different between Norway and the rest of mainland Europe, as Norway had a mean occurrence corresponding to a low occurrence (4.1%), when calculating with data based on the traditional method from 2006, 2009, 2011, 2014 and 2016 [@Kaspersen2018]. In comparison, the mean of the whole of Europe was at 59.3% from 2011 to 2016, excluding 2015 [@EFSA2013;@EFSA2014a;@EFSA2015;@EFSA2016;@EFSA2017;@EFSA2018], which corresponds to a very high occurrence.

![Mean occurrence of quinolone resistance in indicator E. coli from 2011 - 2017. Mean values from broilers was based on data from 2011, 2012, 2013, 2014, and 2016, for pigs from 2011, 2012, 2013, 2015 and 2017, for calves from 2015 and 2017, and for turkeys from 2014 and 2016. Grey areas represent missing data. Data taken from European Union summary reports on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food 2011 - 2017 [@EFSA2013,@EFSA2014a,@EFSA2015,@EFSA2016,@EFSA2017,@EFSA2018,@EFSA2019]. Illustration: H. Kaspersen. \label{fig:ecdcData}](images/ecdcdata2.png)

When looking at the results from indicator _E. coli_ in NORM-VET, the level of QREC occurrence in various animal species in Norway is low. Broilers and wild birds seem to have the highest occurrence [@NORM/NORM-VET2016]. A very low occurrence of QREC has been detected in sheep [@NORM/NORM-VET2018], but none has been detected in horses [@NORM/NORM-VET2009;@NORM/NORM-VET2017] or reindeer [@NORM/NORM-VET2012].



\newpage

<!-- Chapter 6 -->
## Genomic Analysis
The process of analysing genomic data can be compared to piecing together shredded paper - the genomic DNA is fragmented, sequenced, then put together again. Bioinformatics can be defined as the use of informatics techniques, such as mathematics, statistics and computer science, to understand biological data on a large scale [@Luscombe2001]. In this chapter, the concept of high throughput sequencing (HTS) and gene identification is presented, followed by the analysis pathway from reads to assembly. Lastly, various subsequent analyses is briefly introduced.

### High throughput sequencing
The process of Illumina HTS builds upon that of Sanger sequencing. First, genomic DNA is extracted and purified to adequate quality. The genomic DNA (gDNA) is then fragmented and enriched either by PCR or by hybridization [@Morey2013], through a process called library preparation. Adapters are then placed on each end of each DNA fragment, followed by amplification. Then, the prepared fragments are placed on a flow-cell and the sequencing begins. The fragments are polymerized to a given length, usually between 100 - 300 base pairs [@Illumina2019], by incorporating fluorescence labelled bases, which are registered by the instrument. Following sequencing, the reads can be used in a multitude of analyses, briefly summarised in Figure \@ref(fig:bioinfo).

![Main analysis pathway for culture-based genomic data, modified from Springer Nature [@Boolchandani2019]. \label{fig:bioinfo}](images/bioinfo.jpg)


### Computational identification of genes
Computational identification of genes may be done in two ways, either with assemblies or with raw reads. In both cases, a database containing the genes of interest is used. In assembly based methods, e.g. Basic Local Alignment Search Tool (BLAST) [@Altschul1990], the assembly is compared to the references by alignment. In read based methods, e.g. Antimicrobial Resistance Identification By Assembly (ARIBA) [@Hunt2017], the raw reads are mapped to the references. Figure \@ref(fig:mapping) presents the read-based method using the software ARIBA [@Hunt2017].

![Overview of the ARIBA mapping and targeted assembly pipeline, reprinted from Microbiology Society [@Hunt2017]. \label{fig:mapping}](images/mapping.png)


### From reads to assembly
Reconstructing genomes is an important part of gene identification and comparative genomics. Having a plausible reconstruction of each bacterial genome opens up the possibility of comparing genomic elements between isolates, and therefore makes it possible to deduce the relationship between them. This section briefly describes the analysis process from quality control of reads to the final assemblies.

Quality control of nucleotides in sequencing reads is important to make sure that the underlying data is of adequate quality for assembly. Quality parameters in fastq files, which is the common format for Illumina data, can be checked with software like fastQC (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Reads are trimmed to ensure that only high-quality nucleotides are included, and that residual adapter sequences from the sequencing reaction are removed. After trimming, the reads are ready to be assembled.

Before delving into the details of genome assembly, it is important to note that genome assemblies are only plausible reconstructions of the original genomes in the cells from which the DNA was extracted, and is therefore only a computed approximation of the "real" genome. Genome assembly is the process where sequence reads are put together into longer, contiguous sequences called contigs, based on overlapping sequences in the reads. Most assemblers in use for Illumina data today are based on de Bruijn graph algorithms [@Simpson2015]. De Bruijn graph algorithms is based on separating the reads into _k_-mers of a specific length, then creating a graph by identifying overlaps. Then, the algorithm "walks" through the graph, and identifies the optimal path where each edge in the graph is visited only once, also known as an Eulerian path [@Pevzner2001], see Figure \@ref(fig:debruijn). This optimal path then becomes the assembly sequence. Each contiguous sequence of assembled reads is called a contig.

![The two de Bruijn graph based assembly methods for genome assembly: Hamiltonian cycles and Eulerian cycles. Reprinted from Nature Biotechnology [@Compeau2011]. \label{fig:debruijn}](images/debruijn.jpg)


### Post-assembly analyses
Here, the process of gene annotation, pan-genome analysis, and genome alignment is briefly described.

Genome annotation is the process where relevant features in the genome is identified and labelled [@Richardson2013]. First, genes are predicted from the assemblies using software that identifies coding regions, such as open reading frames (ORFs). Then, the predicted genes are compared to a database of known genes and the most significant match is identified by alignment methods. The annotation of this match is then transferred to the predicted gene [@Richardson2013;@Seemann2014]. Annotating the genome enables the identification of core genes, i.e. the genes that are present in at least 99% of all genomes included in the analysis [@Page2015]. This is done with a pan-genome analysis, where all genes in all included genomes are counted and classified based on their percentage presence within all the genomes. Identifying core genes is useful, as it highlights which genes may be highly conserved in the included genomes. An alignment of core genes can subsequently be used to analyse the genetic relationship between the isolates. In an alignment the sequences are assumed to be homologous, and each sequence is aligned to one another into columns. Each column contain a single nucleotide, where most of the nucleotides are homologous. The alignment therefore contains blocks of highly similar sequences, also called locally co-linear blocks (LCBs), and gaps where sequences are either missing or highly divergent. To be able to determine patterns of change between sequences, homologous sequences need to be aligned to be able to compare differences between them correctly [@Higgins2009]. Therefore, aligning sequences is the first step in a phylogenetic analysis, i.e. identifying the evolutionary history of the isolates [@Biology-Online2019].

\newpage

<!-- Chapter 7 -->
## Phylogenetic analysis {#phylo}
A frequent problem encountered by bacteriologists is to identify the relationship between isolates. Very closely related isolates are sometimes called clones. A clone does not have a clear definition, but can be loosely defined as isolates that with high probability originated from a recent common ancestor. However, the definition depends on the method used and previous knowledge about the circumstances in which the isolates were involved. Here, the different aspects of a phylogenetic analysis and the underlying assumptions are briefly described.

### Phylogenetics
Phylogeny is defined as the evolutionary history of a species or organism. Phylogenetics is therefore the analysis of genetic material for the purpose of identifying relatedness between isolates [@Biology-Online2019]. Historically, phylogenetic analysis was done on single gene sequences. Today, it is common to use the genetic elements present in all included taxa, i.e. the core genome. The most common way of representing phylogenetic relationships between isolates is a phylogenetic tree, see Figure \@ref(fig:popstructure). In a phylogenetic tree, the tips, or "leaves", of the tree represent each taxa (isolate). Every node in the tree represent a change between the taxa. The most common method of creating these trees is the "character-state" method, where each position in the alignment (the "character") is independently analysed in regard to which nucleotide is present in that location (the "state"). The nodes in the tree then represent different nucleotide changes, usually single nucleotide polymorphisms (SNPs) or indels (insertions or deletions of one or more nucleotide) in the taxa. SNPs can arise in two ways, either by transitions ($A\,\to\,T$, $G\,\to\,C$, and vice versa) or by transversions ($A\,\to\,G$, $A\,\to\,C$, $T\,\to\,G$, $T\,\to\,C$, and vice versa). Transitions seem to be more frequent than transversions, where transversions seem more disruptive to the gene function because of a more radical change in biochemical properties [@Vandamme2009]. The different mutation rates between sites, also called the rate heterogeneity, are important for the underlying rate of evolution, which may be different between isolates. Therefore, assumptions made on the evolutionary relationship between isolates must use a model of evolution to control for the changes that cannot be seen in the data directly. For example, if Adenine (A) is observed in the data, it is unknown whether it was a transition or transversion that led to the A, or if it was multiple steps, compared to the other isolates in the analysis. Therefore, a mathematical model is needed to make predetermined assumptions about the underlying rate of evolution. Many models exists, where the simplest models assume that any nucleotide has the same probability to be replaced by any other nucleotide [@Jukes1969], whereas more complex models assume different heterogeneity rates and occurrences of all four nucleotides [@Tavare1986].

Recombination, as previously mentioned, does not seem to affect tree topology for _E. coli_, but it does have an effect on branch lengths [@Denamur2010a]. Identifying and removing recombinant areas is therefore important, not only to prevent overestimated branch lengths, but also to ensure that only vertically inherited parts of the genome is included in the analysis. This removal also prevents the inclusion of recombinant areas in the genome which may have a different evolutionary history.

When generating a phylogenetic tree, not just one possible tree can describe the underlying data. Multiple tree structures, or tree topologies, may be generated, and ways to identify the best tree is needed. This is usually handled by using an optimality search criterion method, where multiple trees are generated and the tree with the "best fit" to the data is selected. Maximum likelihood (ML) methods are often implemented to handle these problems. ML methods identify the best tree by searching for the tree that maximizes the probability of observing the character states, given the selected model of evolution [@Vandamme2009]. There may be millions of possible tree structures, and generating trees with the character-state method is computationally demanding. Therefore, ML algorithms apply a heuristic approach, where a "good enough" tree is identified, instead of finding the absolute optimal tree [@Vandamme2009]. However, this does not guarantee that the optimal tree is found, but analysis of simulated data shows that ML methods very often reconstruct the correct tree [@Kuhner1994].