# Introduction

## _Escherichia coli_

### General bacteriology and ecology
The genus _Escherichia_ has been known since 1985 [@Farmer1985], but it was Theodor Escherich that first identified, in 1985, the species _Bacterium coli_, later called _Escherichia coli_ [@Escherich1989]. _Escherichia coli_ (_E. coli_) is the most studied microorganism on earth [@Vila2016], and is often used as a model organism in various microbiology studies and surveillance. _E. coli_ is a gram-negative, rod-shaped, facultative anaerobic bacteria often found in the gastrointestinal tract of warm-blooded animals and humans. There, it may be present as a commensal or as a pathogen. The presence of _E. coli_ in the gastrointestinal tract has been attributed to the relationship between body mass and gut transit times [@Gordon2003], and the probability of detecting _E. coli_ increases with the body size of the host. Thus, the prevalence of _E. coli_ is higher in larger mammals. Also, gut morphology seem to play a role in the prevalence of _E. coli_ [@OBrien2011].

### Population structure
The _E. coli_ population is generally regarded to have a clonal structure [@Tenaillon2010]. Multiple stable lineages, also known as clones, have been found to exist within the _E. coli_ species, among which very little recombination occurs. These stable lineages are known as phylogroups, specifically A, B1, B2, C, D, E, and F (Figure \@ref(fig:popstructure)), based on the presence and absence of specific genes [@Clermont2013]. The molecular variability within these lineages have been postulated to be neutral in terms of organism fitness, and thus not affected by natural selection [@Selander2007].

```{r, fig.cap="\\label{fig:popstructure}Phylogenetic history of \\textit{Escherichia coli}, reprinted with permission from Springer Nature \\citep{Tenaillon2010}", out.width="14cm", out.height="14cm", echo=FALSE, message=FALSE, fig.align='center'}
knitr::include_graphics("images/pop_structure.png")
```

## Monitoring antimicrobial resistance
As part of the Norwegian government's strategy plan against antimicrobial resistance, the Norwegian monitoring programme for antimicrobial resistance in feed, food and animals (NORM-VET) was implemented in 2000. This programme has since 2014 followed the recommendations for surveillance as given by the EU (2013/652/EU) in parallel with national surveillance needs. NORM-VET reports antimicrobial usage as well as resistance, and the data generated serves as a baseline for specific research projects. Yearly reports are published in collaboration with NORM, which monitors antimicrobial resistance in pathogenic bacteria from humans.

## Quinolones and Fluoroquinolones
Quinolones are synthetic antimicrobials that were introduced to the market in the late 1960s [@Emmerson2003]. The first quinolone, Nalidixic acid, was discovered as a byproduct of chloroquine synthesis [@Lesher1962], and was clinically in use in 1967. A few decades later, the synthesis of novel quinolones increased the effectiveness of nalidixic acid towards gram-positive bacteria, and reduced its toxicity by altering its chemical structure. This structural change involved, among others, the addition of a fluoro-group to the molecule, which also increased its potency towards _Enterobacteriaceae_ [@Emmerson2003]. This also lead to a change in nomenclature for these compounds, as the molecules that harbored this structural change now were called fluoroquinolones.
Ciprofloxacin, a well known fluoroquinolone, is often used in human medicine, and have been listed on World Health Organization (WHO) list of essential medicines up until today [@WHO2015]. In veterinary medicine, the fluoroquinolone enrofloxacin is regularly used.

The main target for quinolones and fluoroquinolones (hereafter called quinolones) in _E. coli_, is the type II topoisomerases DNA gyrase (also known as topoisomerase II) and topoisomerase IV [@Drlica1997]. The type II topoisomerases regulate DNA topology by cutting DNA strands, passing another DNA strand through the break, and then seal the break [@Hooper1998]. Both DNA gyrase and Topoisomerase IV is comprised of two subunits, A and B. These are encoded by the genes _gyrA_ and _gyrB_ for DNA gyrase, and _parC_ and _parE_ for topoisomerase IV. The type II topoisomerases are the main target for quinolones in _E. coli_, especially the DNA gyrase. Quinolones bind to these enzymes through a water-metal ion bridge, connecting the quinolone molecule to two specific amino acids; ser 83 and asp 87 in DNA gyrase, and ser 80 and glu 84 in topoisomerase IV [@Aldred2014]. This binding inhibits the ligation function of the enzyme, and fragments the bacterial chromosome, leading to cell death.

## Quinolone resistance mechanisms in _E. coli_

Multiple mechanisms that infer resistance towards quinolones have been discovered in _E. coli_, and they are generally divided into four categories: chromosomal mutations, reduced membrane permeability, efflux pump over expression, and plasmid mediated quinolone resistance (PMQR)[@Correia2017,@Aldred2014a], see Figure \@ref(fig:mechanisms).

```{r, fig.cap="\\label{fig:mechanisms}Quinolone resistance mechanisms, reprinted with permission from Microbiology Society \\citep{Correia2017}", out.width="14cm", out.height="10cm", echo=FALSE, message=FALSE, fig.pos="H"}
knitr::include_graphics("images/mechanisms.jpg")
```

### Chromosomal mutations
Perhaps the most known resistance mechanism towards quinolones is mutations in the genes that code for the target molecules, _gyrA_, _gyrB_, _parC_, and _parE_. Missense mutations in these genes that lead to a substitution of critical amino acids needed for the binding of quinolones to the target molecules may lead to an increased tolerance towards these compounds. In _E. coli_, substitutions in a stretch of amino acids known as the "quinolone resistance determining region" (QRDR) in these four genes is highly correlated with quinolone resistance. In GyrA, this region is between amino acid (AA) 67 to 106 [@Yoshida1991], in GyrB between AA 333 and 481, ParC between AA 51 and 170, and ParE between AA 366 and 523. <!--need reference here! --> Specific substitutions within these areas are often identified in QREC, e.g. the serine 83 and aspartic acid 87 positions in GyrA [@Barnard2001], or the serine 80 position in ParC.

Specific mutations in RNA polymerase subunit B (_rpoB_) have been shown to increase the expression of MdtK, which is a multidrug efflux transporter, resulting in an increase in levofloxacin MIC values [@Pietsch2017a]. 

Favourable mutations, such as those conferring antimicrobial resistance, are relatively rare. However, some bacteria are hypermutators with an elevated mutation rate [@Gross1981]. This elevated mutation rate may be due to the presence of mutator alleles, which can accelerate adaptation and increase the chance for a favorable mutation [@Taddei1997]. Mutations in mutator genes such as _mutS_, _mutL_, and _uvrD_ have been shown to increase MIC-values towards ciprofloxacin in _Pseudomonas aeruginosa_ [@Wiegand2008]. These genes code for proteins that either corrects mismatched DNA produced by multiple mechanisms [@Modrich1989], or have proof-reading activity.

### Reduced membrane permeability
Porin channels in the bacterial membrane are essential for the entry of quinolones into the cell [@Correia2017]. Specific chromosomal regulons are involved in the expression of these membrane proteins, exemplified by the _mar_, _soxRS_ and _rob_ regulons [@Correia2017]. Mutations in these genes may decrease the expression of the Omp-type porins, or prevent _ompF_ mRNA expression, which in turn reduces the drug concentration in the cell [@Oethinger1998,@Cohen1989,@Koutsolioutsou2005,@Amabile-Cuevas1991,@White1997]. Mutations in the regulator gene _soxR_ may alter the expression of _soxS_, which then may regulate the expression of _micF_ in response to superoxide stress, which prevents the _ompF_ mRNA expression [@Amabile-Cuevas1991].

### Efflux pump overexpression
In _E. coli_, the AcrAB-TolC efflux pump have been identified as a major facilitator of quinolone efflux [@Sato2013,@Weston2017]. Mutations in specific regulatory genes such as _marR_ may activate _acrAB_ and _tolC_, thus increasing their expression [@Alekshun1999a]. This will in turn reduce the drug concentration in the cell.

### Plasmid mediated quinolone resistance
Some proteins that infer resistance to quinolones are transcribed from plasmids, such as the quinolone resistance proteins (Qnr), the efflux pumps OqxAB and QepA, and the acetyltransferase Aac(6')-ib-cr.

The Qnr proteins are pentapeptide proteins that are capable of protecting the DNA gyrase and/or topoisomerase IV from quinolone action. The Qnr protein binds to the topoisomerase and prevents the quinolone from stabilizing the lethal gyrase-DNA-quinolone cleavage complex [@Tran2005,@Tran2005a,@Robicsek2006], thus effectively reducing the possible targets for the quinolones. Multiple subtypes of the Qnr proteins have been identified, namely QnrA, QnrB, QnrC, QnrD, QnrS, and QnrVC [@Strahilevitz2009], based on their amino acid sequence. The origin of these proteins is mostly unknown, but variants of the _qnrA_ gene have been identified in the chromosome of _Shewanella algae_, a water-dwelling bacterium, suggesting a reservoir for these proteins [@Poirel2005b].

The efflux pump OqxAB have been shown to increase resistance towards chloramphenicol and quinolones, and to be dependent on the host TolC outer membrane protein in _E. coli_ [@Hansen2004,@Hansen2007]. Similarly, QepA have been found to increase resistance towards ciprofloxacin, norfloxacin and enrofloxacin due to efflux [@Yamane2007]. Interestingly, since quinolones are synthetic antimicrobials, innate transporters for quinolones were not expected to exist in nature. However, it has been speculated that these pumps have evolved as a response to molecules with a structural similarity to quinolones [@Yamane2007].

The acetyltransferase Aac(6')-ib-cr infers resistance towards quinolones by altering the quinolone molecule itself. This compromised structure reduces the activity of the quinolone [@Robicsek2006a]. In general this leads to a low-level quinolone resistance. 

## Epidemiology of quinolone resistant _E. coli_
QREC from a variety of animals have been monitored in NORM-VET since 2006<!--sjekk med AM-->. The relative occurrences of QREC within these animals can be found in Figure \@ref(fig:qrecEpi). When analysing NORM-VET data through the years, broilers seem to have the highest occurrence of QREC when compared to other species [@Kaspersen2018]. However, the occurrence is generally low.

```{r, fig.cap="\\label{fig:qrecEpi}Percent occurrence of QREC per animal species", out.width="12cm", out.height="12cm", echo=FALSE, message=FALSE, fig.pos="H"}
knitr::include_graphics("images/qrec_epi.png")
```


## Genomic Analysis
<!-- intro to bioinformatics -->
<!-- standard software -->
<!-- databases -->

Ever since the eve of high-throughput sequencing platforms in the early 2000s, bioinformatics have become increasingly needed to handle the massive amount of generated data. Bioinformatics can be defined as the use of informatics techniques, such as mathematics, statistics and computer science, to understand biological data on a large scale [@Luscombe2001]. In most cases, DNA or protein sequences represent the biological data, but gene expression data can also be analysed.

A myriad of bioinformatic software have been and are being created. However, some programs have become the gold standard for many bioinformaticians. Such programs include, but are not limited to:

- fastQC: Raw read quality control (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
- Trimmomatic: Trim sequence data [@Bolger2014]
- SPAdes: Genome assembly [@Bankevich2012]
- Prokka: Genome annotation [@Seemann2014]
- Roary: Pan genome analysis [@Page2015]
