# Introduction

<!-- Chapter 1 -->
## Antimicrobial resistance
Antimicrobial resistance is a global threat to human and animal health alike, and the World Health Organization (WHO) urged for a "One Health" approach when publishing the Global Action Plan on antimicrobial resistance (AMR) in 2015 [@WHO2015a]. Multiple definitions on antimicrobial resistance exist based on different criteria and settings. In a clinical setting, antimicrobial resistance is defined as the ability of microorganisms to change in response to the presence of antimicrobials or other factors. This change makes the microorganism able to survive antimicrobial treatment [@WHO2018]. In a laboratory setting, the microorganism is regarded as resistant if it grows in the presence of antimicrobials at a higher concentration than phylogenetically related strains [@Guardabassi2006a]. The discovery of antimicrobials in the early 20th century revolutionized medicine, and the discovery of Penicillin in 1928 by Sir Alexander Fleming and subsequent production in the late 1940s was the turning point for treating bacterial infections. However, already in 1945, Alexander Fleming warned about the mis-use of antimicrobials, stating that exposing bacteria to sub-lethal concentrations in the lab rendered them resistant, and that this also may happen in the body [@Fleming1945].

> *"It is not difficult to make microbes resistant to penicillin in the laboratory by exposing them to concentrations not sufficient to kill them, and the same thing occasionally happened in the body"*
>
> `r tufte::quote_footer("--- Sir Alexander Fleming, Nobel lecture, 1945")`

Systematic misuse of antimicrobials have been attributed to the increased emergence of AMR [@WHO2015a]. To tackle this problem, the World Health Organization (WHO) has implemented a Global Action Plan in 2015 to increase knowledge about AMR and provide guidance for the optimal use of antimicrobial agents [@WHO2015a]. Exposing bacteria to sub-lethal concentrations of antimicrobial compounds is known to generate resistant phenotypes. However, resistance to antimicrobial compounds have been found in prehistoric microbial samples [@Dcosta2011], suggesting that antimicrobial resistance is a natural phenomenon. Other factors, such as environmental stress factors, is also linked to the development of AMR in the absence of antimicrobial selective pressure [@McMahon2007,@Poole2012].




<!-- Chapter 2 -->
## Quinolones and Quinolone resistance mechanisms
### Quinolones
Numerous antimicrobial compounds have been discovered the last 80 years. Some of these compounds were created synthetically, such as quinolones. Quinolones were introduced to the market in the late 1960s [@Emmerson2003]. The first quinolone, Nalidixic acid, was discovered as a byproduct of chloroquine synthesis [@Lesher1962], and was clinically in use in 1967. A few decades later, the synthesis of novel quinolones increased the effectiveness of nalidixic acid towards gram-positive bacteria, and reduced its toxicity by altering its chemical structure. This structural change involved, among others, the addition of a fluoro-group to the molecule, which also increased its potency towards _Enterobacteriaceae_ [@Emmerson2003]. This also lead to a change in nomenclature for these compounds, as the molecules that harbored this structural change now were called fluoroquinolones.
Ciprofloxacin, a well known fluoroquinolone, is often used in human medicine, and have been listed on World Health Organization (WHO) list of essential medicines up until today [@WHO2015]. In veterinary medicine, the fluoroquinolone enrofloxacin is regularly used.

The main target for quinolones and fluoroquinolones (hereafter called quinolones) in _E. coli_, is the type II topoisomerases DNA gyrase (also known as topoisomerase II) and topoisomerase IV [@Drlica1997]. The type II topoisomerases regulate DNA topology by cutting DNA strands, passing another DNA strand through the break, and then seal the break [@Hooper1998]. Both DNA gyrase and Topoisomerase IV is comprised of two subunits, A and B. These are encoded by the genes _gyrA_ and _gyrB_ for DNA gyrase, and _parC_ and _parE_ for topoisomerase IV. The type II topoisomerases are the main target for quinolones in _E. coli_, especially the DNA gyrase. Quinolones bind to these enzymes through a water-metal ion bridge (Figure \@ref(fig:watermetalion)), connecting the quinolone molecule to two specific amino acids; ser 83 and asp 87 in DNA gyrase, and ser 80 and glu 84 in topoisomerase IV [@Aldred2014]. This binding inhibits the ligation function of the enzyme, and fragments the bacterial chromosome, leading to cell death.

```{r, fig.cap="\\label{fig:watermetalion}Overview of the water-metal ion bridge that mediates the quinolone-topoisomerase IV interaction, reprinted with permission from the American Chemical Society \\citep{Aldred2014}.", out.width="8cm", out.height="8cm", echo=FALSE, message=FALSE, fig.align='center'}
knitr::include_graphics("images/watermetalion.jpg")
```

### Quinolone resistance mechanisms in _E. coli_
Multiple mechanisms that infer resistance towards quinolones have been discovered in _E. coli_, and they are generally divided into four categories: chromosomal mutations, reduced membrane permeability, efflux pump over expression, and plasmid mediated quinolone resistance (PMQR)[@Correia2017,@Aldred2014a], see Figure \@ref(fig:mechanisms). This is the traditional way of grouping quinolone resistance mechanisms. However, the usual underlying mechanism is mutations in the chromosome, that either lead to structural changes or changes in expression through various pathways. The four categories are therefore described as two different categories below: chromosomal mutations and PMQR.

```{r, fig.cap="\\label{fig:mechanisms}Quinolone resistance mechanisms, grouped into the traditional categories. Reprinted with permission from Microbiology Society \\citep{Correia2017}.", out.width="14cm", out.height="10cm", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center"}
knitr::include_graphics("images/mechanisms.jpg")
```

#### Chromosomal mutations
Perhaps the most known resistance mechanism towards quinolones is mutations in the genes that code for the target molecules, _gyrA_, _gyrB_, _parC_, and _parE_. Missense mutations in these genes that lead to a substitution of critical amino acids needed for the binding of quinolones to the target molecules may lead to an increased tolerance towards these compounds. In _E. coli_, substitutions in a stretch of amino acids known as the "quinolone resistance determining region" (QRDR) in these four genes is highly correlated with quinolone resistance. In GyrA, this region is between amino acid (AA) 67 to 106 [@Yoshida1991], in GyrB between AA 333 and 481, ParC between AA 51 and 170, and ParE between AA 366 and 523. Specific substitutions within these areas are often identified in QREC, e.g. the serine 83 and aspartic acid 87 positions in GyrA [@Barnard2001], or the serine 80 position in ParC. Mutations in one type II enzyme usually confers a 10 fold increase or less in drug resistance, while mutations in both is often found in isolates with 10-100 fold increase in drug resistance [@Aldred2014a].

Favourable mutations, such as those conferring antimicrobial resistance, are relatively rare. However, some bacteria are hypermutators with an elevated mutation rate [@Gross1981]. This elevated mutation rate may be due to the presence of mutator alleles, which can accelerate adaptation and increase the chance for a favorable mutation [@Taddei1997]. Mutations in mutator genes such as _mutS_, _mutL_, and _uvrD_ have been shown to increase MIC-values towards ciprofloxacin in _Pseudomonas aeruginosa_ [@Wiegand2008]. These genes code for proteins that either corrects mismatched DNA produced by multiple mechanisms [@Modrich1989], or have proof-reading activity.

Porin channels in the gram negative bacterial membrane are essential for the entry of quinolones into the cell [@Correia2017]. Specific chromosomal regulons are involved in the expression of these membrane proteins, exemplified by the _mar_, _soxRS_ and _rob_ regulons [@Correia2017]. Mutations in these genes may decrease the expression of the Omp-type porins, or prevent _ompF_ mRNA expression, which in turn reduces the drug concentration in the cell [@Oethinger1998,@Cohen1989,@Koutsolioutsou2005,@Amabile-Cuevas1991,@White1997]. Mutations in the regulator gene _soxR_ may alter the expression of _soxS_, which then may regulate the expression of _micF_ in response to superoxide stress, which prevents the _ompF_ mRNA expression [@Amabile-Cuevas1991]. Reduced expression of the membrane porins OpmF, OmpC, OmpD and OmpA have been linked to resistance towards quinolones and other antimicrobials [@Bore2007]. 

In _E. coli_, the AcrAB-TolC efflux pump have been identified as a major facilitator of quinolone efflux [@Sato2013,@Weston2017]. Mutations in specific regulatory genes such as _marR_ may activate _acrAB_ and _tolC_, thus increasing their expression [@Alekshun1999a]. This will in turn reduce the drug concentration in the cell. Certain mutations in RNA polymerase subunit B (_rpoB_) have been shown to increase the expression of MdtK, which is a multidrug efflux transporter, resulting in an increase in levofloxacin MIC values [@Pietsch2017a].

#### Plasmid mediated quinolone resistance
Some proteins that infer resistance to quinolones are transcribed from plasmids, such as the quinolone resistance proteins (Qnr), the efflux pumps OqxAB and QepA, and the acetyltransferase Aac(6')-ib-cr.

The Qnr proteins are pentapeptide proteins that are capable of protecting the DNA gyrase and/or topoisomerase IV from quinolone action. The Qnr protein binds to the topoisomerase and prevents the quinolone from stabilizing the lethal gyrase-DNA-quinolone cleavage complex [@Tran2005,@Tran2005a,@Robicsek2006], thus effectively reducing the possible targets for the quinolones. Multiple subtypes of the Qnr proteins have been identified, namely QnrA, QnrB, QnrC, QnrD, QnrS, and QnrVC [@Strahilevitz2009], based on their amino acid sequence. The origin of these proteins is mostly unknown, but variants of the _qnrA_ gene have been identified in the chromosome of _Shewanella algae_, a water-dwelling bacterium, suggesting a reservoir for these proteins [@Poirel2005b]. Other PMQR genes have been identified, such as _mfpA_, _mfpA~MT~_, and _mcbG_. Similar to the _qnr_ genes, _mfpA_ codes for a pentapeptide protein with 18.9% sequence similarity with QnrA [@Strahilevitz2009]. The expression of _mfpA_ seem to confer a four- to eightfold increase in MIC values toward ciprofloxacin [@Montero2001b]. The mechanism by which MfpA confers resistance to quinolones is thought to be similar to the Qnr proteins, by binding to the topoisomerase and effectively protecting them from quinolone action [@Hegde2005]. Similarly, _mcbG_ codes for a pentapeptide repeat protein with 19.6% sequence similarity to QnrA. However, this protein seem to protect the DNA gyrase from the inhibitory action of microcin, which is a peptide antibiotic [@Garrido1988]. The mechanism by which the protein protects DNA gyrase is currently unknown.

The efflux pump OqxAB have been shown to increase resistance towards chloramphenicol and quinolones, and is dependent on the host TolC outer membrane protein in _E. coli_ [@Hansen2004,@Hansen2007]. Similarly, QepA have been found to increase resistance towards ciprofloxacin, norfloxacin and enrofloxacin due to efflux [@Yamane2007]. Interestingly, since quinolones are synthetic antimicrobials, innate transporters for quinolones were not expected to exist in nature. However, it has been speculated that these pumps have evolved as a response to molecules with a structural similarity to quinolones [@Yamane2007].

The acetyltransferase AAC(6')-Ib-cr infers resistance towards quinolones by altering the quinolone molecule itself. The enzyme acetylates the amino nitrogen on the piperazinyl substituent of ciprofloxacin and norfloxacin [@Robicsek2006a]. This compromised structure reduces the activity of the quinolone [@Robicsek2006a]. The AAC(6')-Ib-cr enzyme is a variant of the AAC(6')-Ib proteins, with the specific substitutions W102R and D179Y that are necessary for quinolone resistance [@Robicsek2006a]. Expression of this protein usually leads to a relatively low MIC-value, approximately three- to fourfold increase towards ciprofloxacin and norfloxacin [@Strahilevitz2009]. The increase in MIC conferred by this enzyme is generally lower than the Qnr proteins, mostly due to substrate specificity.




<!-- Chapter 3 -->
## _Escherichia coli_
### General bacteriology and ecology
The genus _Escherichia_ has been known since 1985 [@Farmer1985], but it was Theodor Escherich that first identified, in 1985, the species _Bacterium coli_, later called _Escherichia coli_ [@Escherich1989]. _E. coli_ is the most studied microorganism on earth [@Vila2016], and is often used as a model organism in various microbiology studies and surveillance. _E. coli_ is a gram-negative, rod-shaped, facultative anaerobic bacteria often found in the gastrointestinal tract of warm-blooded animals and humans. There, it may be present as a commensal or as a pathogen.

### Population structure
Four decades of research on the population structure dynamics of _E. coli_ populations have yielded many hypotheses regarding their origin and evolution. In 1973, Milkman studied the electrophoretic mobility of five enzymes in different _E. coli_ strains [@Milkman1973]. Milkman hypothesized that there should be many electrophoretic variants of an enzyme if the different alleles were selectively neutral. In his experiments, he saw little variation in the five different loci, and interpreted this as evidence for selection. Throughout the next ten years, multiple studies suggested that _E. coli_ populations evolved clonally, and that recombination only played a small role in the observed diversity [@Selander1987, @Ochman1984a, @Selander2007]. Specific clonal groups, A, B1, B2, C, D and E, was suggested by Selander in 1987 [@Selander1987], and the molecular variability within these lineages have been postulated to be neutral in terms of organism fitness, and thus not affected by natural selection [@Selander2007]. The presence of these phylogenetic groups was further supported by Herzer in 1990 [@Herzer1990], but only for group A, B1, B2 and D. After the introduction of multi locus sequence typing (MLST) in 1998 [@Maiden1998], multiple studies supported the clonal evolution of _E. coli_ [@Escobar-Paramo2004, @Reid2000] by using different MLST schemes. By using yet another MLST scheme, Wirth et al. analysed 462 _E. coli_ isolates. The results did not support the clonal grouping of _E. coli_, as recombinant sites was purged from the data before a topology similar to previous studies was observed, suggesting that recombination was frequent [@Wirth2006]. However, this was difficult to reconcile, as previous results with MLST supported the clonal evolution of _E. coli_ [@Escobar-Paramo2004, @Reid2000, Chaudhuri2012a]. At the eve of high throughput sequencing (HTS), it was possible to sequence the whole of the _E. coli_ genome, providing discriminatory power unseen in previous methods. Touchon et al. showed, by using phylogeny derived from the core genome, that the results supported the clonal groups suggested in earlier studies [@Touchon2009]. However, the validity of this analysis depends on the extent of homologous recombination in the core genome between lineages [@Chaudhuri2012a].

Nomenclature of the phylogenetic groups identified in the studies described above have been sightly differing, depending on the results of the different studies. In some cases, phylogroup D is paraphyletic, split into D1 and D2 [@Chaudhuri2010], Figure \@ref(fig:popstructure). Clades outside the regular phylogenetic groups were identified by using extended MLST, including 22 housekeeping genes [@Walk2009]. These were referred to as Cryptic Clades, some of which were thought to be more abundant outside the host gastrointestinal tract [@Walk2009]. 

Looking at the results from four decades of research on the population structure and evolution of _E. coli_, it is striking that the phylogenetic groups first identified with methods using only a few loci are still evident with methods using the core genome of _E. coli_ isolates [@Chaudhuri2012a]. Whole genome sequencing (WGS) of _E. coli_ has now been considered as the gold standard for phylogenetic inference, as recombination does not seem to disrupt the phylogenetic signal significantly [@Touchon2009]. In contrast, phylogenetic inference based on a set of limited genes, i.e. MLST, is more susceptible to the impact of recombination, and conclusions based on these should therefore be treated with care [@Chaudhuri2012a].

```{r, fig.cap="\\label{fig:popstructure}Phylogenetic groups of E. coli, based on maximum likelihood analysis of a concatenated alignment of 2173 genes, reprinted from \\citep{Chaudhuri2012a}.", out.width="14cm", out.height="14cm", echo=FALSE, message=FALSE, fig.align='center'}
knitr::include_graphics("images/pop_structure_chaudhuri.png")
```




<!-- Chapter 4 -->
## Norwegian husbandry
### Broiler production
The Norwegian broiler production is a pyramidal structure with purebred animals at the top, breeding animals in the middle, and broilers at the bottom [@Mo2016a]. Eggs are imported from grandparent animals in Sweden which originate mostly from Scotland or Germany. The eggs are hatched in Norwegian hatcheries, and become parent animals. These are reared for around 18 weeks, and are subsequently used for breeding of broiler chickens until around 60 weeks old. The broiler chickens are hatched in hatcheries, and distributed to broiler farms all over the country. Only a few hatcheries exist in Norway. Broiler chickens are reared for 28 - 32 days, depending on weight, then slaughtered [@Nortura2019]. A maximum of 36 kg of chickens are allowed per square meter [@Nortura2019]. The broiler farms have high biosecurity, and is closed to the outside environment.

### Swine production
Swine production in Norway also has a pyramidal structure with unidirectional flow of animals, similar to the broiler production [@Kristoffersen2016]. However, the import of live pigs from other countries to the commercial pig population is negligible [@Animalia2018]. The genetic nucleus herds, where the purebred grand parent animals are produced, are at the top of the pyramid. Hybrid parent animals are produced in multiplier herds, either by recruiting purebred animals from nucleus herds or having a self-recruitment strategy. Gilts from the multiplier herds are distributed to piglet producing herds (either farrow to finish or farrow to grower herds). Additionally, sow pool systems exists in Norway. In the central units of the sow pools, sows are inseminated and kept for the majority of the gestation period up until approx. three weeks before farrowing. At this time, the sows are moved to satellite units of the sow pool system where they are kept before being moved back to the central unit after the piglets have been weaned. The piglet producing herds have a mean weaning age of 33 days, followed by a growing stage until the grower pig is approx. 30 kg and 10-11 weeks of age. The grower pig is then transferred to a specialized fattening pig farm or fattening unit at the same farm (Carl Andreas Grøntvedt, personal communication, 19.08.2019). Specific concession limits regulates the herd sizes of the different production categories, i.e. the maximum amount of animals allowed at a farm at any given time.




<!-- Chapter 5 -->
## Monitoring AMR and QREC epidemiology
### Antimicrobial resistance surveillance
Harmonized monitoring of AMR in commensal _E. coli_ from food and food-producing animals has since 2014 been obligatory for all EU/EEA member states, as implemented by the European Commission (SANCO/11591/2012). The commission describes the sampling frame and analysis of isolates. Commensal _E. coli_ is usually isolated from caecal samples, and antimicrobial susceptibility testing (AST) is carried out using broth microdilution. With this method, the minimum inhibitory concentration (MIC) value is determined for multiple antimicrobials. A MIC value is defined as the lowest concentration of the antimicrobial needed to inhibit growth of a microorganism [@Andrews2001]. Certain thresholds have been defined when categorizing isolates from AST results, based on MIC values. Usually, microorganisms are defined as sensitive (wild type), intermediate, or resistant. These definitions all rely on the MIC values to their respective antimicrobials, based on guidelines from the European Committe on Antimicrobial Susceptibility Testing (EUCAST, www.eucast.org). If resistance is acheieved by a single step, for example by chromosomal mutation or acquisition of a plasmid, the isolate may be expected to fall into two groups, either sensitive or resistant. However, if resistance is achieved in a step-wise fashion, the isolate may fall somewhere in between these two groups [@Bywater2006]. The wild-type population is assumed to not have any acquired or mutational resistance [@Food2016]. To separate the isolates that show a degree of resistance from the wild-type, the epidemiological cut-off value (ECOFF) was implemented. The ECOFF is defined as the MIC value above the distribution of MIC values for the wild-type isolates, see Figure \@ref(fig:ecoff). It is important to distinguish between resistant isolates and intermediate isolates, as the intermediate isolates may still respond in a clinical setting. However, monitoring this group of isolates is important, as they may easily become resistant with the acquisition of additional resistance mechanisms.

In this thesis, all _E. coli_ isolates are defined as resistant if they have a MIC value above the ECOFF.

```{r, fig.cap="\\label{fig:ecoff}Distribution of MIC-values for ciprofloxacin in E. coli. Data: www.eucast.org.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center"}
knitr::include_graphics("images/micdist.jpg")
```

Guidelines from EFSA and ECDC specify the nomenclature of the occurrence of AMR in a population [@Food2016], which is often reported as a percentage. These percentages are usually converted to relative terms like "low" and "high". The percentage values that correspond to these relative terms are "rare" (< 0.1%), "very low" (0.1 - 1%), "low" (> 1 - 10%), "moderate" (> 10 - 20%), "high" (> 20 - 50%), "very high" (> 50 - 70%), and "extremely high" (> 70%).

### The Norwegian monitoring programme {#norwmonitor}
As part of the Norwegian government's strategy plan against antimicrobial resistance, the Norwegian monitoring programme for antimicrobial resistance in feed, food and animals (NORM-VET) was implemented in 2000. The National Institute of Nutrition and Seafood Research, the Norwegian Food Safety Authority, and the Norwegian Veterinary Institute are all participants in this programme. The programme has since 2014 followed the recommendations for surveillance as given by the EU (2013/652/EU) in parallel with national surveillance needs. NORM-VET reports antimicrobial usage as well as resistance, and the data generated serves as a baseline for specific research projects. Yearly reports are published (https://www.vetinst.no/en/surveillance-programmes/norm-norm-vet-report) in collaboration with NORM, which monitors antimicrobial resistance in pathogenic bacteria in humans. A variety of animal species have been sampled in NORM-VET, including cattle, pigs, dogs, broilers, layers, horses, red foxes, reindeer, sheep, turkeys, and multiple wild bird species. Not all animal species are sampled each year, due to the NORM-VET sample regime. Sampling of at least 60% of the domestic production of animals allows for represetative collection of indicator _E. coli_, as recommended by EFSA [@EFSA2016b]. The sample material is commonly faecal or caecal matter, but meat and boot swab samples are also included some years. In broiler and turkey flocks, 10 samples were collected from each flock, where the samples were pooled before culturing.

Culturing and isolation methods have been used to identify _E. coli_ from the samples collected. These isolates are used as an indicator for the presence of resistance in the sampled animal's microflora. Quinolone resistance in indicator _E. coli_ has been monitored in NORM-VET since 2000. First, samples are plated onto lactose-bromothymol blue agar (pre-2013) or MacConkey agar (2013 until today). Then, a random _E. coli_ colony is subjected to susceptibility testing. In 2014, a selective method was introduced. Here, the samples were plated onto MacConkey agar with 0.06 mg/L ciprofloxacin, which is the ECOFF for ciprofloxacin in _E. coli_ as defined by EUCAST. Similar to the method above, one random colony of _E. coli_ was selected and subjected to susceptibility testing. Susceptibility testing was done by broth microdilution.

Prophylactic use of antimicrobial compounds in Norwegian livestock is prohibited, and is only used to treat diseased animals. The use of quinolones to treat livestock in Norway is very low, see Figure \@ref(fig:quinUsage) [@NORM/NORM-VET2017].

```{r, fig.cap="\\label{fig:quinUsage}Overall sales, in kg active substance, of quinolones and macrolides for therapeutic use in terrestrial livestock animals in Norway in 1993 - 2017, reprinted with permission \\citep{NORM/NORM-VET2017}.", echo=FALSE, message=FALSE, out.width="95%"}
knitr::include_graphics("images/quin_usage.png")
```

### Epidemiology of quinolone resistant _E. coli_
#### In Norway
When looking at the results from indicator _E. coli_ isolated non-selectively in NORM-VET, the level of QREC occurrence in various animal species in Norway is low. However, broilers and wild birds seem to have the highest occurrence [@NORM/NORM-VET2016]. No QREC has been detected in sheep [@NORM/NORM-VET2007,@NORM/NORM-VET2009], horses [@NORM/NORM-VET2009,@NORM/NORM-VET2017]  or reindeer [@NORM/NORM-VET2012], but a very low occurrence was detected in cattle [@NORM/NORM-VET2017]. However, the occurrence of QREC in Norway is low when compared to other European countries, see Figure \@ref(fig:qrecEpi).

```{r, fig.cap="\\label{fig:qrecEpi}Percent occurrence of QREC per animal species. Illustration: H. Kaspersen.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center"}
knitr::include_graphics("images/qrec_epi.png")
```

#### In Europe
Occurrence of QREC in Europe seem to follow a north - south trend, where the countries in the north have the lowest occurrence, while the southern countries have the highest, see Figure \@ref(fig:ecdcData). Broilers seem to be the animal with highest occurrence of QREC of tested animal species, similar to Norway. However, the occurrence is markedly different between Norway and the rest of mainland Europe, as Norway had a mean occurrence at 4.1%, when calculating with data based on the traditional method from 2006, 2009, 2011, 2014 and 2016 [@Kaspersen2018], which is a low occurrence when using the limits set by ECDC and EFSA [@Food2016]. In comparison, the mean of the whole of Europe was at 59.3% from 2011 to 2016, excluding 2015 [@Food2013,@Food2014,@Food2015,@Food2016,@Food2017,@Food2018], which is a very high occurrence [@Food2016].

```{r, fig.cap="\\label{fig:ecdcData}Mean occurrence of quinolone resistance in indicator E. coli from 2011 - 2017. Mean values from broilers was based on data from 2011, 2012, 2013, 2014, and 2016, for pigs from 2011, 2012, 2013, 2015 and 2017, for calves from 2015 and 2017, and for turkeys from 2014 and 2016. Grey areas represent missing data. Data taken from European Union summary reports on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food 2011 - 2017 \\citep{Food2013,Food2014,Food2015,Food2016,Food2017,Food2018,Food2019}. Illustration: H. Kaspersen", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center"}
knitr::include_graphics("images/ecdcdata2.png")
```




<!-- Chapter 6 -->
## Genomic Analysis
The process of analysing genomic data can be compared to piecing together shredded paper - the genomic DNA is fragmented, sequenced, then put together again. Afterwards, the assemblies are analysed in various aspects. Ever since the eve of high-throughput sequencing platforms in the early 2000s, bioinformatics have become increasingly needed to handle the massive amount of generated data. Bioinformatics can be defined as the use of informatics techniques, such as mathematics, statistics and computer science, to understand biological data on a large scale [@Luscombe2001]. In most cases, DNA or protein sequences represent the biological data, and is widely used in phylogenetic analysis, gene identification, and other analyses. In this chapter, the process of genomic analysis is presented in detail.

### High-throughput sequencing
Following the sequencing of the Human Genome in 2004 with Sanger sequencing, a multi-million dollar sequencing initiative formed [@Schloss2008], resulting in the emergence of multiple high-throughput sequencing (HTS) technologies. Throughout the years, Illumina has quickly become the dominant short-read technology by which high-throughtput sequencing have developed. With the MiSeq, HiSeq and NextSeq sequencers, HTS has rapidly become both cheaper and more efficient. The MiSeq sequencer has been readily used for bacterial genomes, due to the relatively fast instrument run time and the small size of the genomes [@Reuter2015]. The strength of this technology is the massive parallellization of the sequencing process, quickly generating gigabytes of data. Other sequencing companies exist, but Illumina is the dominant on the market for short read sequencers. Companies like Pacific Biosciences and Oxford Nanopore have sequencers specifically for long-read technology.

The process of Illumina HTS builds upon that of Sanger sequencing. First, genomic DNA is extracted and purified to adequate quality. The genomic DNA (gDNA) is then fragmented and enriched either by PCR or by hybridization [@Morey2013], through a process called library preparation. Here, the genomic DNA is fragmented by either mechanical shearing or by enzymes. Adapters are then placed on each end of each DNA fragment, then amplified. Then, the prepared fragments are placed on a flowcell coated with a lawn of adapter oligos, and a clonal array is generated by bridge PCR [@Morey2013], see Figure \@ref(fig:seqmet). The DNA fragments are denatured, then bind to the flow cell oligonucleotides by the attached adapters. The free end of the DNA fragment then hybridize to an adjacent oligonucleotide. Then, the polymerase extends the oligonucleotide to form a double-stranded fragment. The fragments are then denatured, resulting in two separate fragments relatively close to each other. This is then repeated to form clusters of fragments on the flowcell. This is done to amplify the signal for the sequencer to detect. Afterwards, sequencing-by-synthesis is performed to identify the bases in each DNA fragment. First, the reverse strands created in the clustering step is washed away. The fragments are then polymerized to a given length by incorporating fluoresence labelled bases, which are registered by the instrument. The length of the polymerization depends on the sequencer and what the user has specified. In most cases, the read length is between 100 - 300 base pairs [@Illumina2019]. When the forward strands have been sequenced, a new bridge PCR takes place, and the forward strands are cleaved off. The reverse strands are then sequenced in the same manner as the forward strands [@Illumina2019]. Because the reverse strands are sequenced after the forward strands, a slightly lower per base quality is expected in the reverse strands.

```{r, fig.cap="\\label{fig:seqmet}Steps in Illumina sequencing; A: Generation of the clonal array by bridge PCR. B: Sequencing by synthesis, which takes place in the clonal clusters created in A. Modified and reprinted with permission from Elsevier \\citep{Morey2013}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="90%"}
knitr::include_graphics("images/seqmet.jpg")
```

### Analysis of sequence reads
Raw reads from the sequencer can be analysed in multiple ways, illustrated in Figure \@ref(fig:bioinfo). Raw reads are quality controlled, to check the quality of the read bases and remove low quality nucleotides. This is usually done by programs such as fastQC (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and Trimmomatic [@Bolger2014]. Afterwards, the reads are used to stitch the genome back together, a process called genome assembly.

```{r, fig.cap="\\label{fig:bioinfo}Main analysis pathway for culture-based genomic data, reprinted and modified with permission from Springer Nature \\citep{Boolchandani2019}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="110%"}
knitr::include_graphics("images/bioinfo.jpg")
```

Before delving into the details of genome assembly, it is important to note that genome assemblies are only plausible reconstructions of the original genomes in the cells from which the DNA was extracted. An assembly is a data structure where reads are grouped into contigs, and contigs into scaffolds, and maps the raw sequence data into a reconstruction of the original genome [@Miller2010].

Genome assembly is a process where the quality controlled reads are put together into longer, contiguous sequences called contigs. This is done on the basis of overlapping sequences in the reads. Multiple approaches to genome assembly have been developed, but the graph-based assembly is most frequently used, especially de Bruijn graph based methods [@Simpson2015].

De Bruijn graph algorithms is based on separating the reads into _k_-mers of a specific length, then creating a graph by identifying overlaps, similar to the greedy algorithm. The _k_-mers should always be of odd length to avoid palindromes [@Miller2010]. Then, the algorithm "walks" through the graph, and identifies the optimal path where each edge in the graph is visited only once, also known as an Eulerian path [@Pevzner2001], see Figure \@ref(fig:debruijn). In reality, however, a perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001].
When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. 

```{r, fig.cap="\\label{fig:debruijn}The two de Bruijn graph based assembly methods for genome assembly: Hamiltonian cycles and Eulerian cycles. Reprinted with permission from Nature Biotechnology \\citep{Compeau2011}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="100%"}
knitr::include_graphics("images/debruijn.jpg")
```




<!-- Chapter 7 -->
## Computational identification of antimicrobial resistance genes
Identification of resistance genes, either presence/absence or variants within specific intrinsic genes, is usually done by read-based methods (mapping) or assembly-based methods (BLAST). Using alignment-based approaches for the detection of AMR genes have been shown to be superior to using assembled sequences [@Clausen2016], as the complicated process of _de novo_ genome assembly may lead to the AMR determinants being missed [@Hunt2017]. In this thesis, the resistance genes and variants were identified directly from the sequencing reads by using the Antimicrobial Resistance Identification By Assembly (ARIBA) tool [@Hunt2017]. Resistance towards quinolones in _E. coli_ is usually related to mutations in intrinsic genes, i.e. genes that are naturally present in _E. coli_ and doesn't confer resistance in its wild-type form. In addition, some PMQR genes are also related to quinolone resistance. Therefore, we needed a tool that could identify both mutations in intrinsic genes and plasmid-mediated genes at high accuracy and with the possibility of specifying which databases to use. ARIBA is a tool that fulfills these criteria. 

ARIBA utilizes multiple tools to identify both acquired resistance genes and resistance-related mutations in intrinsic genes. First, the reference sequences in the specified database is clustered by similarity with CD-HIT [@Huang2010]. Then, the raw reads are mapped to the reference sequences by using minimap [@Li2016]. This produces a set of reads for each reference cluster (Figure \@ref(fig:mapping)). Each set of reads are then assembled independently by using fermi-lite (https://github.com/lh3/fermi-lite), and the closest reference sequence in the cluster is identified for each assembled contig with nucmer [@Kurtz2004]. Then, the assembly is compared to this reference sequence to identify completeness and variants by using nucmer and show-snps [@Kurtz2004]. This information is used to generate a bitwise flag which is used to assess the success of the assembly [@Hunt2017]. The raw reads are then mapped to the respective assemblies with Bowtie2 [@Langmead2012] and possible variants are identified with SAMtools [@Li2009a]. Finally, a report is created with all the resulting data [@Hunt2017].

```{r, fig.cap="\\label{fig:mapping}Overview of the ARIBA mapping and targeted assembly pipeline, reprinted with permission from Microbiology Society \\citep{Hunt2017}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center"}
knitr::include_graphics("images/mapping.png")
```

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel genes may therefore be missed. In the case of mutations in intrinsic genes, all genes of interest in this study was included in the database used.




<!-- Chapter 8 -->
## Clustering and phylogenomic analysis
A frequent problem encountered by the bacteriologist is to classify isolates as clones. A clone does not have a clear definition, but can be loosely defined as isolates that with high probability originated from a recent common ancestor. However, the definition depends on the method used and previous knowledge about the circumstances in which the isolates were involved. For example, in an outbreak situation, one would assume that the isolates involved have a recent common ancestor - this may not be the case when working with isolates without a clear epidemiological link. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and allele-based methods like multi locus sequence typing (MLST) or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as allele-based methods do not take into account the amount of difference between alleles [@Feil2010]. The different methods are described in short below.

### Bacterial sequence typing
MLST was first described by Maiden et al. in 1998 [@Maiden1998]. The method relies on sequencing seven different housekeeping genes, and generating a sequence type (ST) from the combination of the respective gene alleles. Isolates with the same ST is regarded as highly similar. The data generated by MLST is comparable between laboratories [@Achtman2004], and multiple gene schemes have been developed. Perhaps the most used scheme is the one hosted by EnteroBase [@Wirth2006], which includes the _adk_, _fumC_, _gyrB_, _icd_, _mdh_, _recA_, and _purA_ genes. The MLST method have been expanded into cgMLST, which includes the core genes identified in the species of interest and provides a deeper resolution on the relationship between the isolates. In _E. coli_, a scheme that includes 2360 genes have been extensively used [@Alikhan2018]. However, to date, cgMLST is not comparable between laboratories.

### Phylogenetics
A more advanced way of deducing the relationship between isolates is phylogenetic analysis. Phylogeny is defined as the evolutionary history of a species or organism. Phylogenetics is therefore the analysis of genetic material for the purpose of identifying relatedness between isolates [@BiologyOn]. Usually, differences in single nucleotide polymorphisms (SNPs) between isolates are used to calculate the genetic distance between them, but presence/absence of genes or other genetic entities may be used. This genetic distance is then used to generate a phylogenetic tree, a representation of the relationship between them. In addition, models of evolution, including rate heterogeniety and substitution rates, are used.

In a phylogenetic analysis, the first step is to align the sequences in question. 




Finding the optimal tree topology in phylogenetic trees can be a computationally costly task. However, modern algorithms use search heuristics to find the "best" tree [@Nguyen2015].














