# Materials and Methods

Here follows a summary of methods used - for details, see enclosed papers.

<!-- Chapter 1  -->
## Laboratory methods

All _E. coli_ isolates included in this project were isolated through the NORM-VET programme, and the methods used for isolating _E. coli_ and susceptibility testing are described in Section \@ref(norwmonitor). An _E. coli_ was categorized as quinolone resistant if the MIC value for ciprofloxacin or nalidixic acid was above 0.06 mg/L or 16 mg/L, respectively, based on ECOFF values defined by EUCAST (ECOFF values as of 01.08.2019, www.eucast.org).

Library preparation and sequencing service was provided by the Norwegian Sequencing Centre (NSC, www.sequencing.uio.no), a national technology platform hosted by the University of Oslo and supported by the "Functional Genomics" and "Infrastructure" programs of the Research Council of Norway and the Southeastern Regional Health Authorities.

<!-- Chapter 2  -->
## Bioinformatic methods
Bioinformatic analysis was largely performed on the Abel Cluster, owned by the University of Oslo and Uninett/Sigma2, and operated by the Department for Research Computing at USIT, the University of Oslo IT-department (www.hpc.uio.no/).

### Identification of resistance mechanisms {#aribaflag}

Antimicrobial Resistance gene Identification by Assembly (ARIBA) [@Hunt2017] was used to identify resistance genes with the Resfinder [@Zankari2012] database and mutations with the MEGARes database [@Lakin2017], as well as determining the sequence types of the isolates, with the _E. coli_ scheme hosted by Enterobase [@Wirth2006]. The process by which ARIBA works is visualized in Figure \@ref(fig:mapping).

```{r, fig.cap="\\label{fig:mapping}Overview of the ARIBA mapping and targeted assembly pipeline, reprinted from Hunt et al. 2017 \\citep{Hunt2017} with permission from Microbiology Society. Program names are listed in brackets.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="70%",out.height="70%"}
knitr::include_graphics("images/pdfs/mapping.pdf")
```

ARIBA is a read-based algorithm that works by clustering the reference sequences from the selected database (Figure \@ref(fig:mapping)). Then, read pairs are mapped to the clusters and locally assembled. The closest reference to the assembled sequence is identified, and various quality metrics are calculated, such as gene completeness and overall success of the local assembly [@Hunt2017]. These metrics are encoded into flags, which represent the overall quality of the identified gene or mutation. Each flag has a specific interpretation, as exemplified for flag 27 in Table \@ref(tab:flag). In the present study, flags that represented the following were accepted:

\newpage

- The gene is adequately assembled
  + 95% of the reference sequence is identical to the assembled sequence
  + The gene is assembled into only one contig
  + No assembly scaffold graph ambiguity

- No technical errors when identifying closest reference sequence

In total, 16 different flags represented these criteria, and were used to ensure high quality of the predicted genes and variants. An R script was used to select the genes or variants that fulfilled these criteria (www.github.com/hkaspersen/VAMPIR).
\
```{r flag, echo=FALSE}
data.frame("Status" = c("TRUE",
                            "TRUE",
                            "FALSE",
                            "TRUE",
                            "TRUE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE"),
           "Metric" = c("Assembled",
                        "Assembled into one contig",
                        "Region assembled twice",
                        "Complete gene",
                        "Unique contig",
                        "Scaffold graph bad",
                        "Assembly fail",
                        "Variants suggests collapsed repeat",
                        "Hit both strands",
                        "Has variant",
                        "Ref seq choose fail")) %>%
  kable("latex",
        booktabs = TRUE,
        caption = "Overview of flag 27 interpretation") %>%
  kable_styling(latex_option = "HOLD_position")
```

### Assembly, annotation and pan-genome analysis
All software used for assembly, annotation and pan-genome analysis is presented in Figure \@ref(fig:assembly). Major software settings are listed in each box.

```{r, fig.cap="\\label{fig:assembly}Flowchart of assembly and annotation pipeline. Light blue: input data. Yellow rectangle: analyses on the Abel cluster, with important program settings listed. Dark blue ellipses: analyses in R. Green folders: output data.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="90%",out.height="90%"}
knitr::include_graphics("images/pdfs/assembly_pipeline.pdf")
```

### Phylogenetic analysis
Phylogenetic analysis methods are summarized in Figure \@ref(fig:flowchart). The left pathway was used to identify the overall relationship between all isolates in each paper. The right pathway was used to further analyse clades of interest with deeper resolution and recombination removal.
\
\
```{r, fig.cap="\\label{fig:flowchart}Flowchart of phylogenetic methods. Left: phylogenetic analysis for all isolates. Right: phylogenetic analysis for selected clades. Light blue: input data. Yellow rectangle: analyses on the Abel cluster. Dark blue ellipses: analyses in R. Green folders: output data.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="99%"}
knitr::include_graphics("images/pdfs/flowchart.pdf")
```

<!-- Chapter 3 -->
## Statistical methods
Statistical analysis was done in `r version$version.string`[@R-base].
Significant differences between groups was calculated by using $\chi$^2^-test. Confidence intervals were calculated by using the two-sided exact binomial test at 95% confidence level.

A non-parametric permutation test was used to assess significant aggregation of isolates in phylogenetic trees. Here, the mean/median minimum SNP distance to closest isolate within the same animal species (Paper II) or phenotype (Paper III) was calculated. Then, a permutation test was used to calculate the mean/median minimum SNP distance for each iteration (_n_ = 1000). Non-exact p-values were calculated based on the number of expected values that were lower than the observed value for all iterations.

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of quinolone resistance conferring substitutions and genes by using the vegan package [@R-vegan]. NMDS is an ordination technique that has previously been used to relate the presence of acquired AMR genes to sample source or type [@Wang2016;@Liu2019]. A stressplot was calculated to determine how well the ordination represented the data [@R-vegan].





