# Materials and Methods

Here follows a summary of methods used - for details, see enclosed articles.

<!-- Chapter 1  -->
## Laboratory methods

All _E. coli_ isolates included in this study were isolated through the NORM-VET programme, and the methods used for isolating _E. coli_ and susceptibility testing is described in Section \@ref(norwmonitor). An _E. coli_ was categorized as quinolone resistant if the MIC value for ciprofloxacin was above 0.06 mg/L.

DNA was extracted from pure bacterial cultures of the isolates included in article II and III. Library preparation and sequencing service was provided by the Norwegian Sequencing Centre (NSC, www.sequencing.uio.no), a national technology platform hosted by the University of Oslo and supported by the "Functional Genomics" and "Infrastructure" programs of the Research Council of Norway and the Southeastern Regional Health Authorities.

<!-- Chapter 2  -->
## Genomic analysis methods
Parts of the genomic analysis was performed on the Abel Cluster, owned by the University of Oslo and Uninett/Sigma2, and operated by the Department for Research Computing at USIT, the University of Oslo IT-department (www.hpc.uio.no/). The rest of the analyses was performed in R [@R-base].

### Quality control
All raw reads in this study were quality controlled with FastQC [@BabrahamBioinformatics2018]. Possible contaminant reads were identified with Mash screen [@Ondov2016]. Residual phiX was removed with bbduk [@Bushnell2019] by mapping reads to the phiX genome (NC_001422.1). Unmapped sequences were then trimmed with trimmomatic [@Bolger2014].

### Gene and variant detection {#aribaflag}
Antimicrobial Resistance gene Identification by Assembly (ARIBA) [@Hunt2017] was used to determine the sequence types of the isolates, with the _E. coli_ scheme hosted by Enterobase [@Wirth2006]. ARIBA was also used to identify acquired resistance genes with the resfinder database [@Zankari2012], and mutations in intrinsic genes with the MEGARes database [@Lakin2017]. The process by which ARIBA works is visualized in Figure \@ref(fig:mapping).


![Overview of the ARIBA mapping and targeted assembly pipeline, reprinted from Microbiology Society [@Hunt2017]. \label{fig:mapping}](images/mapping.png)


ARIBA is a read-based algorithm that works by first clustering the reference sequences from the selected database (Figure \@ref(fig:mapping)). Then, read pairs are mapped to the clusters and locally assembled. Then, the closest reference to the assembled sequence is identified, and various quality metrics are calculated, such as gene completeness and overall success of the local assembly [@Hunt2017]. These metrics are encoded into flags, which represent the overall quality of the identified gene or mutation. Each flag has a specific interpretation, as exemplified for flag 27 in Table \@ref(tab:flag). In the present study, flags that represented the following were accepted:

- The gene is adequately assembled

- The gene is assembled into one contig

- No regions in the assemblies are assembled twice

- No assembly scaffold ambiguity

- Adequate read depth

- No errors when identifying closest reference sequence

In total, 16 different flags represented these criteria, and were used to ensure high quality of the predicted genes and variants. An R script was used to select the genes or variants that fulfilled these criteria (www.github.com/hkaspersen/VAMPIR).

```{r flag, echo=FALSE}
data.frame("Status" = c("TRUE",
                            "TRUE",
                            "FALSE",
                            "TRUE",
                            "TRUE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE",
                            "FALSE"),
           "Metric" = c("Assembled",
                        "Assembled into one contig",
                        "Region assembled twice",
                        "Complete gene",
                        "Unique contig",
                        "Scaffold graph bad",
                        "Assembly fail",
                        "Variants suggests collapsed repeat",
                        "Hit both strands",
                        "Has variant",
                        "Ref seq choose fail")) %>%
  kable("latex",
        booktabs = TRUE,
        caption = "Overview of flag 27 interpretation") %>%
  kable_styling(latex_option = "HOLD_position")
```


The mutator genes _dam, mutD, mutH, mutL, mutM, mutS, mutT, mutY_, and _uvrD_ were investigated in regards to mutations that lead to amino acid changes in the encoded proteins. The sequences were retrieved from each isolate using BLAST [@Altschul1990]. Mutations were detected by comparing to the _E. coli_ K12 versions of the genes.

### Assembly, annotation and pan-genome analysis
The process of assembly and annotation is visualized in Figure \@ref(fig:assembly).

![Flowchart of assembly and annotation pipeline. Yellow rectangle: Analysis on the Abel cluster, with important program settings listed. Turquoise ellipses: Analyses in R. Green folders: Output data. Illustration: H. Kaspersen. \label{fig:assembly}](images/assembly_pipeline.png)

SPAdes [@Bankevich2012] was used to assemble the genomes from the trimmed reads, filtering out contigs smaller than 500 bp. Pilon [@Walker2014] was used to correct potential errors that arose in the assembly process. Prokka [@Seemann2014] was used to annotate the pilon-corrected genomes. Five well-annotated, curated and complete genomes of _E. coli_ was used as a reference to ensure high quality of the annotations. Quast [@Gurevich2013] was used to assess the error-corrected assemblies. A pan-genome analysis was done using Roary [@Page2015], which was also used to create a core gene alignment. 

### Phylogenetic analysis
Phylogenetic analysis methods are summarized in Figure \@ref(fig:flowchart). In article II, the left pathway in Figure \@ref(fig:flowchart) was used to generate an overview tree of all the isolates. Then, clades were selected for further analysis based on low genetic divergence, animal species, and geographical location. These isolate groups were then analysed with the rightmost pathway. In article III, only the left pathway was used, excluding the patristic distances. 

![Flowchart of phylogenetic methods. Left: Phylogenetic analysis for all isolates. Right: Phylogenetic analysis for selected clades. Purple folders represent input data. Beige boxes represent bioinformatic tools used in Linux. Turquoise circles represent analyses in R. Yellow folders represent output data. Illustration: H. Kaspersen \label{fig:flowchart}](images/flowchart.png)

<!-- Chapter 3 -->
## Statistical methods
Statistical analysis was done in R version 3.4.2 (Article 1) and `r version$version.string` (Article 2 and 3) [@R-base].
Significant differences between groups was calculated by using $\chi$^2^-test. Confidence intervals were calculated by using the two-sided exact binomial test at 95% confidence level.

A non-parametric permutation test was used to assess significant aggregation of isolates in phylogenetic trees. Here, the mean/median minimum SNP distance to closest isolate within the same animal species (Article II) or phenotype (Article III) was calculated. Then, a permutation test was used to calculate the mean/median minimum SNP distance for each iteration (_n_ = 1000). Non-exact p-values were calculated based on the number of expected values that were lower than the observed value for all iterations.

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of quinolone resistance conferring substitutions and genes by using the vegan package [@R-vegan]. NMDS is an ordination technique that has previously been used to relate the presence of acquired AMR genes to sample source or type [@Wang2016;@Liu2019]. A stressplot was calculated to determine how well the ordination represented the data [@R-vegan].





