# Materials and Methods

Here follows a summary of methods used - for details, see enclosed articles.

<!-- Chapter 1  -->
## Laboratory methods

### Isolate information
All _E. coli_ isolates included in this study were isolated through the NORM-VET programme, and the methods used for isolating _E. coli_ and susceptibility testing is described in Section \@ref(norwmonitor).

### DNA isolation and sequencing
QREC isolates were plated onto MacConkey agar with 0.06 mg/L ciprofloxacin to confirm resistance. DNA was isolated from pure bacterial cultures using the QIAmp DNA mini kit (Qiagen). DNA quality was measured on a NanoDrop^TM^ One spectrophotometer (Thermo Scientific), and DNA concentration was measured with a broad-range DNA Qubit assay (Qiagen). DNA integrity was assessed by gel electrophoresis and/or on a Fragment Analyser^TM^ Automated CE System Instrument (FSV2-DE2-100, Advanced Analytical).

The Nextera XT/Flex (Illumina) library preparation kits were used for preparing the libraries for sequencing. Sequencing was done on Illumina HiSeq platforms, either Hiseq 2000, Hiseq 2500 rapid run, Hiseq 3/4000, or Hiseq X. The sequencing and library preparation service was provided by the Norwegian Sequencing Centre (NSC, www.sequencing.uio.no), a national technology platform hosted by the University of Oslo and supported by the "Functional Genomics" and "Infrastructure" programs of the Research Council of Norway and the Southeastern Regional Health Authorities. Both library preparation and sequencing was performed by NSC.

<!-- Chapter 2  -->
## Genomic analysis methods
### Quality control, genome assembly and annotation
All raw reads in this study were quality controlled with FastQC (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Possible contaminant reads were identified with Mash screen [@Ondov2016]. Residual phiX was removed with bbduk (https://jgi.doe.gov/data-and-tools/bbtools/) by mapping reads to the phiX genome (NC_001422.1). Unmapped sequences were then trimmed with trimmomatic [@Bolger2014]. SPAdes [@Bankevich2012] was used to assemble the genomes, and Pilon [@Walker2014] was subsequently used for error correction. Prokka [@Seemann2014] was used to annotate the pilon-corrected genomes. 

### AMR gene identification and MLST {#aribaflag}
ARIBA was used to determine the sequence types of the isolates, with the _E. coli_ scheme hosted by Enterobase [@Wirth2006].
ARIBA was also used to identify acquired resistance genes with the resfinder database [@Zankari2012], and mutations in intrinsic genes with the MEGARes database [@Lakin2017]. All ARIBA flags which fulfilled the following criteria was accepted (see https://github.com/sanger-pathogens/ariba/wiki/Task:-flag for details):

- At least 95% of the reference sequence matches the assembly on only one contig

- No ambiguity in the scaffolding graph


### Phylogenetic analysis
A core gene alignment was generated with Roary [@Page2015], and constant sites was removed using snp-sites [@Page2016]. IQTree [@Nguyen2015] was used to generate a ML-tree from the resulting alignment, with ultrafast bootstrap approximation and model finder plus with the ascertainment bias correction (ASC). 

Deeper phylogenetic analysis was done on selected clades with high similarity. First, the core genome was determined by using ParSNP [@Treangen2014]. Harvesttools [@Treangen2014] was used for file conversion. Then, Gubbins [@Croucher2015] was used to detect and remove recombinant sites. IQTree was subsequently used on the recombinant-filtered alignment to create a ML-tree, using the same settings as above.


<!-- Chapter 3 -->
## Statistical methods
Statistical analysis was done in R version 3.4.2 (Article 1) and `r version$version.string` (Article 2 and 3) [@R-base].
Significant differences between groups was calculated by using $\chi$^2^-test. Confidence intervals were calculated by using the two-sided exact binomial test at 95% confidence level.

Relatedness was determined by calculating pairwise distances between isolates. To test if individuals within animal groups were more closely related than randomly expected, the individuals within animal groups were permuted in 1000 iterations. Percentage of distances below 0.1 and above 0.8 was calculated in each iteration. Non-exact p-values was then calculated on the basis of how many expected values from x iterations were above the observed value. 

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of quinolone resistance conferring substitutions and genes, with the vegan package [@R-vegan], with 200 random starts. A stressplot was used to determine goodness of fit.