# Materials and Methods

Here follows a summary of methods used - for details, see enclosed papers.

<!-- Chapter 1  -->
## Laboratory methods

All _E. coli_ isolates included in this project were isolated through the NORM-VET programme, and the methods used for isolating _E. coli_ and susceptibility testing are described in Section \@ref(norwmonitor). An _E. coli_ was categorized as quinolone resistant if the MIC value for ciprofloxacin or nalidixic acid was above 0.06 mg/L or 16 mg/L, respectively, based on ECOFF values defined by EUCAST (ECOFF values as of 01.08.2019, www.eucast.org).

Library preparation and sequencing service was provided by the Norwegian Sequencing Centre (NSC, www.sequencing.uio.no), a national technology platform hosted by the University of Oslo and supported by the "Functional Genomics" and "Infrastructure" programmes of the Research Council of Norway and the South-eastern Regional Health Authorities.

<!-- Chapter 2  -->
## Bioinformatic methods
Bioinformatic analysis was mainly performed on the Abel Cluster, owned by the University of Oslo and Uninett/Sigma2, and operated by the Department for Research Computing at USIT, the University of Oslo IT-department (www.hpc.uio.no/).

### Identification of resistance mechanisms {#aribaflag}

The program ARIBA (Antimicrobial Resistance gene Identification by Assembly) [@Hunt2017] was used to identify resistance genes with the Resfinder [@Zankari2012] database and mutations with the MEGARes database [@Lakin2017], as well as determining the sequence types of the isolates, with the _E. coli_ scheme hosted by Enterobase [@Wirth2006]. The process by which ARIBA works is visualized in Figure \@ref(fig:mapping).

```{r, fig.cap="\\label{fig:mapping}Overview of the ARIBA mapping and targeted assembly pipeline, reprinted from Hunt et al. 2017 \\citep{Hunt2017} with permission from Microbiology Society. Program names are listed in brackets.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="80%",out.height="80%"}
knitr::include_graphics("images/pdfs/mapping.pdf")
```

ARIBA is a read-based algorithm that works by clustering the reference sequences from the selected database (Figure \@ref(fig:mapping)). Then, reads and their pairs (mate in Figure \@ref(fig:mapping)) are mapped to the clusters and locally assembled. The closest reference to the assembled sequence is identified, and 11 different quality metrics are calculated, such as gene completeness and overall success of the local assembly [@Hunt2017]. The combination of all metrics is encoded into a flag, which is a number given by ARIBA based on the underlying set of quality metrics for each predicted gene or mutation. Only one flag is given for each predicted mutation or gene, and each flag has a specific interpretation. In the current study, three of the 11 metrics were evaluated as TRUE, while four were evaluated as FALSE to ensure high quality of the predicted mutation or gene. The remaining four metrics could be either TRUE or FALSE. All allowed combinations of metrics are listed in Table \@ref(tab:flag). These criteria resulted in 16 different flags that were accepted. All genes or mutations that were predicted with a non-accepted flag were removed. An R script was used to select the genes or mutations that fulfilled these criteria (www.github.com/hkaspersen/VAMPIR).
\
```{r flag, echo=FALSE}
data.frame(
  "Metric" = c(
    "95% of the reference sequence is identical to the assembly",
    "The gene is assembled into only one contig",
    "< 3% of the reference has more than one match to the assembly",
    "The gene is complete (from start to stop)",
    "Exactly one contig in the assembly matches the reference",
    "Assembly graph ambiguity",
    "No output from assembler",
    "Variant in position that matches to the reference",
    "Two or more matches to the reference in opposite orientations",
    "Variant present",
    "Error when selecting closest reference"
  ),
  "Value" = c(
    "TRUE",
    "TRUE",
    "FALSE",
    "TRUE | FALSE",
    "TRUE",
    "FALSE",
    "FALSE",
    "TRUE | FALSE",
    "TRUE | FALSE",
    "TRUE | FALSE",
    "FALSE"
  )
) %>%
  kable("latex",
        booktabs = TRUE,
        caption = "All metrics reported by ARIBA and their allowed values used in this study. All possible combinations of these metrics resulted in 16 different allowed flags. See github.com/sanger-pathogens/ariba/wiki/Task:-flag for a detailed description of each metric.") %>%
  kable_styling(latex_option = "HOLD_position",
                full_width = FALSE,
                font_size = 10) %>%
  pack_rows("Assembled", 1, 1) %>%
  pack_rows("Assembled into one contig", 2, 2) %>%
  pack_rows("Region assembled twice", 3, 3) %>%
  pack_rows("Complete gene", 4,4) %>%
  pack_rows("Unique contig", 5,5) %>%
  pack_rows("Scaffold graph bad", 6,6) %>%
  pack_rows("Assembly fail", 7,7) %>%
  pack_rows("Variant suggests collapsed repeat", 8,8) %>%
  pack_rows("Hit both strands", 9,9) %>%
  pack_rows("Has variant", 10,10) %>%
  pack_rows("Ref seq choose fail", 11,11)
```

\newpage

### Assembly, annotation and pan-genome analysis
All software used for assembly, annotation and pan-genome analysis is presented in Figure \@ref(fig:assembly).

```{r, fig.cap="\\label{fig:assembly}Flowchart of assembly and annotation pipeline. Light blue: input data. Yellow rectangle: name of the analysis tool used on the Abel cluster, with important program settings listed. Dark blue ellipses: analyses in R. Green folders: output data.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="85%",out.height="85%"}
knitr::include_graphics("images/pdfs/assembly_pipeline.pdf")
```

### Phylogenetic analysis
Phylogenetic analysis methods are summarised in Figure \@ref(fig:flowchart). The left pathway was used to identify the overall relationship between all isolates in each paper. The right pathway was used to further analyse clades of interest with deeper resolution and recombination removal.
\
\
```{r, fig.cap="\\label{fig:flowchart}Flowchart of phylogenetic methods. Left: phylogenetic analysis for all isolates. Right: phylogenetic analysis for selected clades. Light blue: input data. Yellow rectangle: name of the analysis tool used on the Abel cluster. Dark blue ellipses: analyses in R. Green folders: output data.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="100%", out.height="100%"}
knitr::include_graphics("images/pdfs/flowchart.pdf")
```

<!-- Chapter 3 -->
## Statistical methods
Statistical analysis was done in `r version$version.string`[@R-base].
Significant differences between groups were calculated by using $\chi$^2^-test. Confidence intervals were calculated by using the two-sided exact binomial test at 95% confidence level.

A non-parametric permutation test was used to assess significant aggregation of isolates in phylogenetic trees. Here, the median minimum SNP distance to the closest isolate within the same animal species was calculated. Then, a permutation test was used to calculate the median minimum SNP distance for each iteration (_n_ = 1000). Non-exact p-values were calculated based on the number of expected values that were lower than the observed value for all iterations.

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of quinolone resistance conferring substitutions and genes by using the vegan package [@R-vegan]. NMDS is an ordination technique that has previously been used to relate the presence of acquired AMR genes to sample source or type [@Wang2016;@Liu2019]. A stress plot (Shepard diagram) was calculated to determine how well the ordination represented the data [@R-vegan].
