# Discussion

## Methodological considerations

### Selection bias

In the epidemiological study (paper I), the census population of _E. coli_ and QREC obtained through the NORM-VET programme from 2006 to 2016 were included. The sampling performed in NORM-VET from livestock animals at slaughterhouses is based on a proportional sampling according to the slaughter volume at each slaughterhouse. The sampling is done at a random week of the year, with only one sample per flock or herd. These samples can be regarded as representative for the livestock populations in Norway. For the wild animals, the sampling is dependent on hunters or other people sending in carcasses to be included in the programme. Therefore, these samples may not be randomly distributed throughout Norway, even with efforts to request samples from the whole country. Regardless, the isolate collection from the wild animals is likely as representative for the actual _E. coli_ population in the respective animal species as possible, and were regarded as such in the epidemiological study.

In paper II, a random selection of isolates was performed after grouping the isolates on phenotypic resistance patterns. This was done to ensure a high phenotypic diversity among the selected isolates, to be able to cover all available phenotypic combinations and subsequently detect the highest diversity of resistance mechanisms among the QREC isolates. This selection therefore aimed to maximise phenotypic diversity among the isolates, and may not represent the actual QREC populations in each included animal species. 

In paper III, the isolate selection included both QREC and quinolone susceptible (wild type) _E. coli_. Here, the goal was to further investigate dissemination, as well as to identify possible development of quinolone resistance in wild type isolates. Isolates were included if the production site was sampled at least three times, and at least one QREC isolate and one wild type isolate were detected in samples from the respective production site. The results may therefore be biased because of the uneven representation of each production site. In addition, not all production sites were represented.

### Genomic analyses methods
The various software used in the genomic analysis were selected because they are known and reliable, and most of them are implemented in the in-house pipeline.

#### Contaminants
MALDI-TOF was used to confirm the species as _E. coli_ during DNA extraction. However, some samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. Inoculating impure samples may have resulted in extraction of DNA from multiple species, as some bacteria may have similar morphology on MacConkey media. This contamination resulted in the exclusion of four isolates due to high amounts of reads originating from these species. Reads from other bacterial species than _E. coli_ may also have been present in sequencing reads from other isolates. However, filtering contigs smaller than 500 bp after assembly should remove most of the contaminant contigs, as they are most likely small in size due to a low amount of reads present from these contaminants.

#### Identification of genes and mutations
In this project, ARIBA [@Hunt2017] was used to identify resistance genes, mutations in intrinsic genes related to resistance, and for sequence typing. For this work, ARIBA has multiple advantages, such as being able to detect both resistance genes and mutations. ARIBA is a read-based program, and using a read-based approach for the detection of AMR genes has been found to be superior to using assembled sequences [@Clausen2016]. The process of genome assembly is complicated, and if one gene is for example split over multiple contigs they may be missed [@Hunt2017;@Clausen2016]. Another advantage of using ARIBA is the possibility of downloading newly updated databases of our choosing. Here, the ResFinder [@Zankari2012] database was used for identification of acquired genes, and the MegaRes [@Lakin2017] database for mutations in chromosomal genes. Resfinder is a popular database for acquired resistance genes, which is manually curated and updated regularly. MEGARes was originally created for metagenomic characterization of AMR genes and mutations [@Lakin2017], and contains references from ResFinder, ARG-ANNOT, the Comprehensive Antibiotic Resistance Database (CARD), in addition to the National Center for Biotechnology Information (NCBI) Lahey clinic $\beta$-lactamase archive. MEGARes is also manually curated, and is specifically annotated for high-throughput data processing [@Lakin2017]. MEGARes was chosen as a database because it already encompasses other popular resistance databases, and is easily downloaded through ARIBA.

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel acquired genes are therefore missed. However, some mutations or genes may have been misclassified as not present due to the handling of the flags reported by ARIBA. ARIBA provide flags to assess the quality of the identified gene or mutation. In the present study, flags that did not represent the inclusion criteria were excluded from the final results. Thus, some isolates may have been categorized as negative for the given variant or gene, when in reality the gene or variant were not of adequate quality to be included. In this project, only novel mutations were confirmed by identifying the same mutation in the respective assemblies. This was not done, however, for previously known mutations. One way of correcting for possible false negatives from the ARIBA results would be to confirm with assembly based methods, such as PointFinder [@CGE2019].

### Phylogeny
Phylogenetic methods were used to identify the evolutionary relationship between the isolates because the isolates had no predefined epidemiological link. In an outbreak situation, it is to be assumed that the isolates involved have a recent common ancestor - this was nt the case among the isolates included in this project. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and typing-based methods like MLST or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as typing-based methods do not take into account the amount of difference between alleles [@Feil2010].

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, which may have influenced the phylogenetic analysis. However, Pilon was used to correct potential errors in the assembly process by mapping the reads back to the assembly. Thus, sequencing errors were likely corrected.

Removing recombinant sites is an important step when analysing vertical inheritance. Here, Gubbins was used in this regard, which have been found to be much faster than similar software [@Croucher2015], such as ClonalFrameML [@Didelot2015].

IQTree was selected as the ML algorithm to find the optimal phylogenetic tree. Comparative analyses have shown that the search strategy implemented in IQTree achieves higher likelihoods than similar algorithms [@Nguyen2015], such as RAxML [@Stamatakis2014] and PhyML [@Guindon2010], and IQTree seem to find optimal trees faster. Additionally, IQTree is easier to use for non-evolutionary biologists compared to RAxML, which was the main incentive in selecting IQTree. However, IQTree was not always the best algorithm in the comparative analysis between RAxML, PhyML and IQTree, and the authors recommended using all three programs [@Nguyen2015]. In this study, only IQTree was used, which means that potential trees with higher likelihoods may have been missed. 

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree for each phylogenetic analysis. ModelFinder finds the optimal evolutionary model for the data, based on criteria that control for overparameterization [@Kalyaanamoorthy2017]. This way, the data itself choose which model is appropriate, rather than the user. However, using more than one model may be necessary when working with many isolates from potentially different lineages, as some lineages may evolve differently than others. The selected models used in this study may therefore not be optimal in describing the evolutionary rate of all the isolates in one tree. This is also partly why deeper phylogenetic analysis was performed on more closely related isolates separately, such as the ST117 isolates in Article II and ST355 isolates in Article III, as other more appropriate evolutionary models were determined for these trees. 

### Statistical considerations

A non-parametric iteration test was used to assess the aggregation of samples in phylogenetic trees based on SNP-distances. A non-parametric test was selected because the underlying distribution of SNP-distances did not follow a theoretical distribution, such as a normal distribution. Additionally, the data could not be regarded as independent observations. Therefore, a test that does not rely on distribution assumptions or independent observations was selected. 

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence absence of resistance genes and mutations (Article II). NMDS is an ordination technique that differs from other ordination methods, such as principal component analysis (PCA), as NMDS does not assume a relationship between the samples [@Holland2008]. Therefore, any distance measure is allowed in NMDS, which makes this method well suited for a wide variety of data. NMDS is an iterative method that finds a solution after _n_ number of iterations or after an acceptable solution has been found. This means that running a separate NMDS analysis on the same data may yield a slightly different result. Therefore, the solution found in paper II may thus represent an adequate solution rather than the "best" one. A stress value is calculated for each iteration and represents how good the data is summarized by the ordination. In the present study, this stress value was low (< 0.05), which is regarded as a good representation [@Buttigieg2014a; @Buttigieg2014].




## Main results and discussion

### Occurrence of QREC
At the start of the project, the only information available on occurrence of QREC in Norwegian animals were in the NORM-VET reports. Here, occurrence data for each year was readily available, but no overall occurrence comparison between animal species across the years had been performed. Therefore, little was known about possible temporal trends within animal species, or the difference in overall occurrence between animal species. Compiling and analysing data from NORM-VET surveillance was therefore regarded as a good starting point, as this may give pointers for where further investigation is needed. The results of this investigation showed an overall low occurrence in several animal species in Norway, both wild and domestic. An increasing trend of QREC occurrence was detected in broilers. The paper concluded that since little to no quinolones are used in these animal species, other factors may play a role in the development of QREC in these animal species.
In the current study, different levels of QREC occurrence were detected with the traditional method among commensal _E. coli_ in different production animal species between 2006 and 2016. These animals are fed different feed, stay in different environments, have a different anatomy, and vastly different production site densities, which may affect the QREC occurrence within these animal populations. No QREC were detected in horses or sheep in the current project. However, QREC were detected in 0.3% among commensal _E. coli_ isolates from sheep in NORM-VET 2018 [@NORM/NORM-VET2018]. An overall QREC occurrence of 0.3% and 0.4% was identified among commensal _E. coli_ from pigs and cattle, respectively. All of these occurrence levels are comparable to the levels reported by other Nordic countries, such as Sweden and Denmark, in the same time frame [@SVARMREPORTS2019; @DANMAPREPORTS2019], see Figure \@ref(fig:nordic). The overall European level of ciprofloxacin resistant _E. coli_ occurrence in pigs and cattle was in 2018 10.6% for both species [@EFSA2017], indicating that the Nordic countries have a much lower occurrence level than most other European countries. For broilers, the occurrence seem to be more fluctuating among the Nordic countries. A slightly increasing trend of QREC occurrence is observed in both Norway and Denmark. Contrastingly, the occurrence in Sweden has been decreasing since 2013 from 14% to 7% in 2018 [@SWEDRES/SVARM2013; @SWEDRES/SVARM2018]. Regardless of these increasing and decreasing trends, the overall occurrence of QREC in the three countries from 2010 to 2018 is relatively similar, at 2.13% in Norway, 2.66% in Denmark, and 2.89% in Sweden based on data from the NORM-VET, SVARM and DANMAP reports [@SVARMREPORTS2019; @DANMAPREPORTS2019], corresponding to an overall low occurrence.

```{r, fig.cap="\\label{fig:nordic}Overall QREC occurrence in broilers, pigs and cattle in Norway, Sweden and Denmark between 2009 and 2018. Data summarized from the NORM-VET, SVARM, and DANMAP reports, respectively \\citep{NORMVETREPORTS2019,SVARMREPORTS2019,DANMAPREPORTS2019}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="110%", out.height="110%"}
knitr::include_graphics("images/pdfs/qrec_occurrence_nordic.pdf")
```

Norway is, to date, the only country that has done selective screening for QREC among selected animal species, and comparison of occurrence data generated by this method to other countries is therefore difficult. However, the selective method supplies valuable data when used in conjunction with the traditional method, as the two methods together provide a rough overview of the relative amount of QREC in the samples tested. For example, using the selective method, an occurrence of 2.4% was detected in samples from horses [@NORM/NORM-VET2017], 7.2% in samples from cattle [@NORM/NORM-VET2015], and 9.6% in samples from sheep [@NORM/NORM-VET2018]. In contrast, the corresponding occurrence in broilers and pigs were 89.5% and 54.3%, respectively [@NORM/NORM-VET2014; @NORM/NORM-VET2015]. Taken together with the occurrences generated with the traditional method presented above, there is an obvious difference in QREC occurrence between cattle, sheep and horses when compared to pigs and broilers. A thought arose during the project that the anatomy of ruminants might affect the occurrence of QREC in these animal species. This hypothesis had previously been addressed in a Swedish study, which concluded that the prevalence of QREC was higher in calves less than 18 days of age compared to older calves [@Duse2015]. Very young veal calves have not yet developed the intestinal microbiota, and are regarded as monogastric up until the animal stops drinking milk from the mother and starts eating solid food. At this point, the animal is regarded as a ruminant. A small pilot study addressing this hypothesis was performed, where the temporal occurrence of QREC in calves and their mothers within two different farms were investigated (unpublished data). The calves were sampled frequently during the first three months of life, and the samples were subjected to selective screening of QREC. The results indicated that there were other factors than age alone that affected the occurrence of QREC. However, as this was only a small pilot study, additional studies are needed to further investigate this hypothesis.

In NORM-VET, pigs are sampled individually and the samples are not pooled together as they are for broilers. The occurrences detected with both methods indicate that QREC is present in a high proportion of samples from pigs, but at low levels. The occurrence of QREC in other Nordic countries is similarly very low [@EFSA2017]. A case control study on the effect of decontamination with quinolones on the occurrence of QREC over time is currently ongoing at the Institute. Here, herds that were previously decontaminated with quinolones were sampled as case herds, and herds where no quinolones had been used were sampled as control herds. A semi-quantitative method was used to identify the occurrence and relative presence of QREC in these samples. The results showed a significant difference between the case and control herds, and gave indications that QREC persisted in the production environment years after decontamination with quinolones (unpublished data). However, it is unknown if these QREC isolates were already present before being exposed to the quinolones, or if commensal isolates developed resistance as a response to the exposure. The findings in this ongoing study may indicate that even with low quinolone usage, QREC may be selected for or develop in the gut of these animals and persist in the pig production environment.

In this study, the occurrence of QREC was decidedly highest in broilers compared to the other production animals included. Samples from broilers are pooled before screening, which may have increased the probability of detecting QREC among the samples tested. Nonetheless, since quinolones are not used in the Norwegian broiler production, a question arose regarding their origin. It was hypothesized that the observed occurrence in broilers was either due to introduction and dissemination of existing strains, or by spontaneous development of quinolone resistance in wild type _E. coli_ in the gut of the animal due to unknown selective pressure. Similarly, because of the observed occurrence of QREC detected with the selective method in pigs, isolates from both species were whole genome sequenced to detect possible dissemination of similar isolates within each production chain. Additionally, QREC isolates from wild animals were included to identify isolates that may be introduced to these production chains from the outside. Moreover, which resistance mechanisms that were dominant among the QREC isolates from these animal populations was not known, thus a resistance mechanism characterization was performed.


### Resistance mechanism characterization and comparative genomics
The results in paper II suggested that the majority of resistance mechanisms identified were chromosomally encoded. The major resistance mechanisms causing quinolone resistance among these isolates were mutations in _gyrA_ and _parC_, which are well described mechanisms in the literature [@Hooper2015]. Regarding the other investigated chromosomal genes, no direct effect of the identified mutations was possible to deduce from the current genomic data. In-vitro mutational studies are needed to further investigate if these mutations have an effect on MIC values toward quinolones. Transcriptomics may also give useful information on the expression levels of these proteins.
A relatively high occurrence of PMQR was detected among the pig isolates (26.6%) compared to the broiler isolates (4.6%). The low occurrence of PMQR in broilers is in concordance with other Nordic studies on QREC [@Myrenaas2018; @Borjesson2015], and may indicate that PMQR determinants are relatively rare in the Nordic broiler production chain. These concordant results emphasize the dominance of chromosomally encoded quinolone resistance mechanisms in the QREC isolates from the broiler production, and is in favour of the hypothesis that QREC isolates are mainly vertically disseminated in the Nordic broiler production chain. Therefore, PMQR seem to be of minor importance in the development and dissemination of QREC in the Nordic broiler production chain. On the other hand, PMQR genes, specifically _qnrB19_  and _qnrS1_, were detected in a high proportion of the pig isolates. _qnrB19_ have previously been identified on small, non-conjugative plasmids harbouring little to no other resistance genes conferring resistance toward other antimicrobials [@Soares2019], while _qnrS1_ has been associated with large, conjugative plasmids harbouring multiple resistance genes [@Dolejska2011a; @Slettemeas2019]. Plasmid characterization has not been done in the current study, but if the _qnrB19_ genes are indeed located on non-conjugative plasmids, then this may indicate that they are clonally disseminated in the pig production chain. 

Phylogenetic analysis of QREC isolates from broilers and pigs revealed possible dissemination within the respective production chains. Moreover, possible persistence in the broiler production chain was detected. Additionally, highly similar major sequence types among isolates from broilers were detected. These findings, together with the low occurrence of PMQR in broilers, suggested a clonal dissemination of QREC in the broiler production chain. The data thus suggested that dissemination was a major contributor to the observed occurrence of QREC in the broiler production chain. However, it was unknown if these strains were disseminated from higher in the production chain as suggested by a Swedish study [@Borjesson2015], or if the resistance developed locally in wild type isolates due to an unknown selective pressure. To investigate this further, both QREC and wild type _E. coli_ was whole genome sequenced and compared (Paper III). 

The results in paper III further supported the hypothesis that QREC is disseminated through the Norwegian broiler production chain, and likely originate from imported breeding animals. Imported breeding animals have been implicated in the introduction of resistant _E. coli_ into the broiler production chain in several Nordic countries, both for QREC and cephalosporin-resistant _E. coli_ [@Myrenaas2018; @Borjesson2015; @Nilsson2014; @Bortolaia2010a; @Petersen2006]. These results, in combination with no quinolone usage in the Norwegian broiler production, provides strong evidence for implicating import of breeding animals as a major causative factor for the occurrence of QREC in the Norwegian broiler production. However, only a few samples from parent flocks were included in this project. To confirm that the major QREC lineages are introduced to the Norwegian broiler production through imported breeding animals, further sampling and characterization of QREC from parent flocks need to be performed. Many studies have focused on the national dissemination of both cephalosporin-resistant and quinolone resistant _E. coli_ in the broiler production chain, where they have provided data that points to implicating imported animals [@Borjesson2015; @Nilsson2014; @Mo2014]. Future studies on these matters should therefore focus on comparing isolates from several countries and thereby investigate these aspects higher in the broiler production pyramid, in addition to studying the temporal effects by screening through several succeeding samplings. To the authors’ knowledge, no data on QREC occurrence or genetic characterization of QREC is available from the grandparent distributor in Scotland or Germany. This information would further provide much needed data on this hypothesized international dissemination.

The possibility of local development of quinolone resistance in wild type _E. coli_ have, to the authors’ knowledge, not previously been performed by comparing QREC and wild type isolates at a phylogenetic level. The result of this analysis in paper III indicated that this rarely occurs in the Norwegian broiler production chain. However, two isolate pairs were identified where this may have occurred, with a considerable amount of time between the isolates' sampling. This indicate a possibility of local selective pressure at the lower part of the broiler production pyramid, which may perhaps be related to exposure to disinfectants or cleaning agents [@Buffet-Bataillon2016]. However, it is likely that this potential selective pressure is not very strong, as only a low level of QREC have been detected among the QREC positive samples. Taken together with the evidence for import and dissemination described above, the development of quinolone resistance in wild type _E. coli_ seem to be of minor importance for the occurrence of QREC in the Norwegian broiler production.

Other than the work presented in paper II, no further investigation of the presence of QREC in the Norwegian pig production was performed in this project. Since the Norwegian pig production is domestic, QREC is less likely to be introduced to the pig production chain in a similar manner as the broiler production, since little to no live pigs are imported each year. Since the QREC isolates from pigs in the present study were not characterized as deeply as the broiler isolates, their origin is yet to be identified. Feed has previously been found to introduce QREC strains to the broiler production chain on Iceland [@Thorsteinsdottir2010], which may also be the case for pig feed in Norway. However, no QREC were identified in pig feed by using the selective method in 2016 [@NORM/NORM-VET2016], which makes it unlikely as a source of QREC in the pig production. Further studies are needed to investigate the source of QREC in the Norwegian pig production chain.
