# Discussion

## Methodological considerations

### Selection bias

In the epidemiological study (paper I), the census population of _E. coli_ and QREC obtained through the NORM-VET programme from 2006 to 2016 were included. The sampling performed in NORM-VET from livestock animals at slaughterhouses is based on a proportional sampling according to the slaughter volume at each slaughterhouse. The sampling happens at a random week during the year which the sampled animal species is included, with only one sample per flock or herd. These samples can be regarded as representative for the livestock populations in Norway. For the wild animals, the sampling is dependent on hunters or other people sending in carcasses to be included in the programme. Therefore, these samples may not be randomly distributed throughout Norway, even with efforts to request samples from the whole country. Regardless, the isolate collection from the wild animals is likely as representative for the actual _E. coli_ population in the respective animal species as possible, and were regarded as such in the epidemiological study.

In paper II, a random selection of isolates was performed after grouping the isolates on phenotypic resistance patterns. This was done to ensure a high phenotypic diversity among the selected isolates, to be able to cover all available phenotypic combinations and subsequently detect the highest diversity of resistance mechanisms among the QREC isolates. This selection is therefore based on maximising phenotypic diversity among the isolates, and may not represent the actual QREC populations in each included animal species. 

In paper III, the isolate selection included both QREC and wild type _E. coli_, where quinolone susceptible _E. coli_ were regarded as wild type. Here, the goal was to investigate possible persistence within a production site, as well as to identify possible development of quinolone resistance in wild type isolates. Isolates were included if the production site was sampled at least three times, and at least one QREC isolate and one wild type isolate were detected in samples from the respective production site. The results may therefore be biased because of the uneven representation of each production site. However, no conclusions regarding production sites were made in the paper.

### Genomic analyses methods
The various software used in the genomic analysis was selected because they are known and reliable, and most of them are implemented in the in-house pipeline.

#### Contaminants
MALDI-TOF was used to confirm the species as _E. coli_ during DNA extraction. However, some samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. Inoculating impure samples may have resulted in extraction of DNA from multiple species, as some bacteria may have similar morphology on MacConkey media. This contamination resulted in the exclusion of four isolates due to high amounts of reads originating from these species. Reads from other bacterial species than _E. coli_ may also have been present in other isolates. However, filtering contigs smaller than 500 bp after assembly should remove most of the contaminant contigs, and increased the overall quality of the assemblies.

#### Identification of genes and mutations
In this project, ARIBA [@Hunt2017] was used to identify resistance genes, mutations in intrinsic genes related to resistance, and for sequence typing. For this work, ARIBA has multiple advantages, such as being able to detect both resistance genes and mutations. ARIBA is a read-based program, and using a read-based approach for the detection of AMR genes has been found to be superior to using assembled sequences [@Clausen2016]. The process of genome assembly is complicated, and if one gene is for example split over multiple contigs they may be missed [@Hunt2017;@Clausen2016]. Another advantage of using ARIBA is the possibility of downloading newly updated databases of our choosing. Here, the ResFinder [@Zankari2012] database was used for identification of acquired genes, and the MegaRes [@Lakin2017] database for mutations in chromosomal genes. Resfinder is a popular database for acquired resistance genes, which is manually curated and updated regularly. MEGARes was originally created for metagenomic characterization of AMR genes and mutations [@Lakin2017], and contains references from ResFinder, ARG-ANNOT, the Comprehensive Antibiotic Resistance Database (CARD), in addition to the National Center for Biotechnology Information (NCBI) Lahey clinic $\beta$-lactamase archive. MEGARes is also manually curated, and is specifically annotated for high-throughput data processing [@Lakin2017]. MEGARes was chosen as a database because it already encompasses other popular resistance databases, and is easily downloaded through ARIBA.

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel acquired genes are therefore missed. However, some mutations or genes may have been misclassified as not present due to the handling of the flags reported by ARIBA. ARIBA provide flags to assess the quality of the identified gene or mutation. In the present study, flags that did not represent the inclusion criteria were excluded from the final results. Thus, some isolates may have been categorized as negative for the given variant or gene, when in reality the gene or variant were not of adequate quality to be included.

### Phylogeny
Phylogenetic methods were used to identify the evolutionary relationship between the isolates. An evolutionary approach was used because the isolates had no clear epidemiological link. In an outbreak situation, it is to be assumed that the isolates involved have a recent common ancestor - this may not be the case in situations where the isolates are not expected to be closely related. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and typing-based methods like MLST or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as typing-based methods do not take into account the amount of difference between alleles [@Feil2010]. Therefore, a phylogenetic approach was used to get a deeper resolution when determining the relationship between isolates.

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, which may have influenced the phylogenetic analysis. Pilon was used to correct errors by mapping the reads back to the assembly. However, even with this correction step, errors may persist and influence the topology or branch lengths in the final trees.

Removing recombinant sites is an important step when analysing vertical inheritance. Here, Gubbins was used in this regard, which have been found to be much faster than similar software [@Croucher2015], such as ClonalFrameML [@Didelot2015].

IQTree was selected as the ML algorithm to find the optimal phylogenetic tree. Comparative analyses have shown that the search strategy implemented in IQTree achieves higher likelihoods than similar algorithms [@Nguyen2015], such as RAxML [@Stamatakis2014] and PhyML [@Guindon2010], and IQTree seem to find optimal trees faster. Additionally, IQTree is easier to use for non-evolutionary biologists compared to RAxML, which was the main incentive in selecting IQTree. However, IQTree was not always the best algorithm in the comparative analysis between RAxML, PhyML and IQTree, and the authors recommended using all three programs [@Nguyen2015]. In this study, only IQTree was used, which means that potential trees with higher likelihoods may have been missed. 

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree for each phylogenetic analysis. ModelFinder finds the optimal evolutionary model for the data, based on criteria that control for overparameterization [@Kalyaanamoorthy2017]. This way, the data itself choose which model is appropriate, rather than the user. However, using more than one model may be necessary when working with many isolates from potentially different lineages, as some lineages may evolve differently than others. The selected models used in this study may therefore not be optimal in describing the evolutionary rate of all the isolates in one tree. This is also partly why deeper phylogenetic analysis was performed on more closely related isolates separately, such as the ST117 isolates in Article II and ST355 isolates in Article III, as other more appropriate evolutionary models were determined for these trees. 

### Statistical considerations

A non-parametric iteration test was used to assess the aggregation of samples in phylogenetic trees based on SNP-distances. A non-parametric test was selected because the underlying distribution of SNP-distances did not follow a theoretical distribution, such as a normal distribution. Additionally, the data could not be regarded as independent observations. Therefore, a test that does not rely on distribution assumptions or independent observations was selected. 

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of resistance genes and mutations (Article II). NMDS is an ordination technique that differs from other ordination methods, such as principal component analysis (PCA), as NMDS does not assume a relationship between the samples [@Holland2008]. Therefore, any distance measure is allowed in NMDS, which makes this method well suited for a wide variety of data. NMDS is an iterative method that finds a solution after _n_ number of iterations or after an acceptable solution has been found. This means that running a separate NMDS analysis on the same data may yield a slightly different result. Therefore, the solution found in paper II may thus represent an adequate solution rather than the "best" one. A stress value is calculated for each iteration and represents how good the data is summarized by the ordination. In the present study, this stress value was low (< 0.05), which is regarded as a good representation [@Buttigieg2014a; @Buttigieg2014].




## Main results and discussion

### Occurrence of QREC
At the start of the project, the only information available on QREC in Norwegian animals were in the NORM-VET reports. Here, occurrence data for each year was readily available, but no overall occurrence comparison between animal species across the years had been performed. Therefore, little was known about possible temporal trends within animal species, or the difference in overall occurrence between animal species. Compiling and analysing data from NORM-VET surveillance was therefore regarded as a good starting point, as this may give pointers for where further investigation is needed. The results of this investigation showed an overall low occurrence in several animal species in Norway, both wild and domestic. An increasing trend of QREC occurrence was detected in broilers. The paper concluded that since little to no quinolones are used in these animal species, other factors may pay a role in the development of QREC in these animal species.

No QREC were detected in horses, reindeer or sheep. However, QREC was detected in 0.3% of the included sheep samples in NORM-VET 2018 [@NORM/NORM-VET2018]. Using the selective method, an occurrence of 2.4% was detected in samples from horses in 2017 [@NORM/NORM-VET2017]. No selective screening of sheep or reindeer have been performed. This low occurrence, even with selective screening in horses, indicate that QREC is rare in these animal species. Reindeer are boreal herd animals that probably have less contact with human waste or other sources of bacteria, which may explain the low occurrence of QREC. A very low to no occurrence of QREC was identified in ruminant species, such as cattle and sheep. A thought arose during the project that the anatomy of ruminants may affect the occurrence of QREC in these animal species, as a higher occurrence of QREC have been reported in veal calves compared to milk cows [@Kaesbohrer2012; @Duse2015]. Very young veal calves have not yet developed the intestinal microbiota, and are regarded as monogastric up until the animal stops drinking milk from the mother and starts eating solid food. At this point, the animal is regarded as a ruminant. A small pilot study addressing this hypothesis was performed, where the temporal occurrence of QREC in calves and their mothers within two different farms were investigated (unpublished data). The results indicated non-age dependent drivers for quinolone resistance, but additional studies are needed to further investigate this hypothesis. 

Different levels of QREC occurrence were detected between the production animal species. These animals are fed different feed, have different lifestyles, anatomy, and production site densities, which may affect dissemination within these animal populations. Furthermore, the production pressure, e.g. the turnover of produced animals, is decidedly highest for broilers than any of the other production animal species. This may partially explain the observed occurrence in broilers, as increased animal contact may contribute to increased chance for QREC infection. A significantly higher occurrence of QREC in broilers and wild birds compared to the other animal species was detected. In wild birds this may not have been an unexpected finding, since wild birds have a high degree of freedom and may be migratory. This allows them to be exposed to other environments, maybe in southern European countries with a much higher overall occurrence of QREC [@EFSA2017; @EFSA2016; @EFSA2015]. However, in broilers, the occurrence could not be attributed to migration or a high degree of movement. The broiler production houses have a high degree of biosecurity, preventing exposure to the outside. Therefore, it was hypothesized that the observed occurrence in broilers was either due to introduction of existing strains from imported breeding animals, as previously hypothesized by Börjesson et al. [@Borjesson2015], or by spontaneous development of quinolone resistance in wild type _E. coli_ in the gut of the animal due to unknown selective pressure. To investigate possible introduction and dissemination, QREC isolates from broilers were whole genome sequenced, along with isolates from pigs, red foxes and wild birds. Additionally, which quinolone resistance mechanisms were dominant or the occurrence of PMQR were unknown among these isolates. Therefore, a characterization of quinolone resistance mechanisms was performed.

### Resistance mechanisms and phylogenetic analysis

In paper II, possible dissemination of QREC was investigated, and resistance mechanisms were characterized in 280 QREC isolates from broiler, pigs, red foxes and wild birds. The results gave indications of dissemination of QREC within the broiler and pig production chains. Additionally, very little PMQR was identified among the broiler isolates, in contrast to wild birds and pigs, where a relatively high occurrence of PMQR was detected.

The major resistance mechanism causing quinolone resistance among these isolates were mutations in _gyrA_ and _parC_, which are well described mechanisms in the literature [@Hooper2015]. Regarding the other chromosomal genes investigated for mutations, no direct effect of these mutations was possible to deduce from the current genomic data. In-vitro mutational studies are needed to further investigate if these mutations have an effect on MIC values toward quinolones. Transciptomics may also give useful information on the expression levels of some of these proteins.

A very low occurrence of PMQR was identified among broiler isolates. This finding was in concordance with other nordic studies on QREC [@Myrenaas2018; @Borjesson2015], and may indicate that PMQR determinants are relatively rare in the Nordic broiler production chain. These concordant results emphasize the importance of chromosomally encoded quinolone resistance mechanisms in the QREC isolates from the broiler production, as these isolates seem to be mainly vertically disseminated. Therefore, PMQR seem to be of minor importance in the development and dissemination of QREC in the Nordic broiler production chain. 

No further investigation of the presence of QREC in the Norwegian pig producton was performed in this project, other than the work presented in paper II. Investigating possible causative factors for the presence of QREC in the pig production is just as important, as the production is domestic. QREC can therefore not be introduced to the pig production chain in a similar manner as the broiler production, since little to no live pigs are imported each year. Interestingly, a relatively high occurrence of PMQR, specifically _qnrB19_, was detected among QREC isolates from pigs. _qnrB19_ have previously been identified on small, non-conjugative plasmids harbouring little to no other resistance genes conferring resistance toward other antimicrobials [@Soares2019]. Plasmid characterization has not been done in the current study, but if the _qnrB19_ genes are indeed located on non-conjugative plasmids, then this may indicate that they are clonally disseminated in the pig production chain. Since the QREC isolates from pigs in the present study were not characterized as deeply as the broiler isolates, their origin is yet to be identified. One possible route of introduction may be pig feed. Feed has previously been found to introduce QREC strains to the broiler production chain on Iceland [@Thorsteinsdottir2010], which may perhaps also be the case for pigs in Norway. However, studies involving feed are needed to investigate these aspects further.

After identifying evidence for both dissemination and persistence in the Norwegian broiler production, it was decided to further investigate this phenomenon by characterizing QREC isolates from broilers separately. The results in paper III further supported the hypothesis that QREC is disseminated through the Norwegian broiler production chain, and likely originate from imported breeding animals. The results also revealed a temporal presence of major STs in the broiler production, suggesting that re-introduction may be a major contributor to the occurrence of QREC in the broiler production rather than persistence. Re-introduction fits with the hypothesis of QREC from imported animals, as parent animals are imported at regular intervals from Sweden. Imported breeding animals have been implicated in the introduction of resistant _E. coli_ into the broiler production chain in several Nordic countries, both for QREC and cephalosporin-resistant _E. coli_ [@Myrenaas2018; @Borjesson2015; @Nilsson2014; @Bortolaia2010a; @Petersen2006]. These results, in combination with no quinolone usage in the Norwegian broiler production, provides strong evidence for implicating import of breeding animals as a major causative factor for the occurrence of QREC in the Norwegian broiler production. However, only a few samples from parent flocks were included in this project. To confirm that the major QREC lineages are introduced to the Norwegian broiler production through imported breeding animals, further sampling and characterization of QREC from parent flocks need to be performed. Many studies have focused on the national dissemination of both cephalosporin-resistant and quinolone resistant _E. coli_ in the broiler production chain, where they have provided data that points to implicating imported animals [@Borjesson2015; @Nilsson2014; @Mo2014]. Future studies on these matters should therefore focus on comparing isolates from several countries, and investigating these aspects higher in the broiler production pyramid. To the authors knowledge, no data on QREC occurrence or genetic characterization of QREC is available from the grandparent distributor in Scotland or Germany. This information would further provide strong evidence of this hypothesized international dissemination.

The possibility of local development of quinolone resistance in wild type _E. coli_ had, to the authors knowledge, not previously been performed by comparing QREC and wild type isolates at a phylogenetic level. The result of this analysis in paper III indicated that this rarely occurs, which was not an unexpected finding due to no known selection pressure for quinolone resistance in the broiler production chain. However, one isolate pair was identified where this may have occurred, with a considerable amount of time between the isolates' sampling. This indicate that there may be some kind of local selective pressure at the lower part of the broiler production pyramid, which is maybe related to exposure to disinfectants or cleaning agents [@Buffet-Bataillon2016]. Nonetheless, taken together with the evidence for import and dissemination described above, the development of quinolone resistance in wild type _E. coli_ seem to be of minor importance for the occurrence of QREC in the Norwegian broiler production. 
