# Discussion
## Main results and discussion
Here, the main results from article I - III is discussed in context of the main aims of the project.

### Summarise and describe existing data on occurrence of quinolone resistance







<!-- Emergence of QREC in multiple animal species in Europe and other countries, even with no quinolone usage -->
<!-- Petersen et al. 2006 -->

Studies have described an increase in QREC occurrence despite low or decreasing use of quinolones [@Petersen2006; @Kijima-Tanaka2003]. Additionally, a significantly increasing trend in ciprofloxacin resistance from 2011 to 2014 was observed in Belgium [@Hanon2015].








Surveillance of quinolone resistance among indicator _E. coli_ has been done in NORM-VET since 2000. Occurrence data were therefore readily available for collation and comparison between screened animal species. The results showed a highest occurrence in broilers and wild birds, followed by red foxes, turkeys and dogs. Furthermore, no association between occurrence of QREC and human population density was found. 







The occurrence of QREC have previously been linked to consumption of quinolones in broiler flocks [@Literak2013a;@Garau1999]. However, use of other antimicrobials may also select for QREC, such as amoxicillin [@Jimenez-Belenguer2016]. 




### Genetic mechanisms that may explain the observed quinolone resistance

Quinolone resistance mechanisms were characterized in article II and III by using ARIBA and by comparing to _E. coli_ K12 versions of mutator genes. Overall, the identified quinolone resistance mechanisms are in concordance with known major mechanisms that cause quinolone resistance in _E. coli_ [@Hooper2015]. The main mechanism by which _E. coli_ gain quinolone resistance is missense mutations in the QRDR of _gyrA_, especially mutations leading to the amino acid substitution S83L. Furthermore, missense mutations in _parC_ and/or _parE_ have often been implicated in conjunction with missense mutations in _gyrA_ [@Lascols2007; @Slettemeas2019; @KompLindgren2003; @Vanni2014; @Zurfluh2014], but rarely on their own. In article II, only two isolates had amino acid substitutions in the QRDR of _ParC_ alone. However, these two isolates harbored the same _qnr_-gene instead. In the article II data, the presence/absence of _qnr_ genes had a strong negative correlation with the presence/absence of amino acid substitutions in the QRDR of GyrA _r_(278) = -0.91, _p_ < 0.05. This suggests either that _qnr_ genes are not often acquired in isolates already harboring missense mutations in _gyrA_, or that mutations in _gyrA_ do not arise so easily in isolates that already harbor _qnr_-genes. A previous study suggested that if _qnr_-genes were present in isolates with missense mutations in the QRDR of _gyrA_, then most likely the mutation preceeded the _qnr_ gene [@Cesaro2008]. This is likely, because the Qnr proteins protect the topoisomerases from quinolone action, and reduces the amount of lethal quinolone-DNA-topoisomerase cleavage complexes in the cell [@Robicsek2006a]. These findings may therefore suggest that the presence of _qnr_-genes prevents mutations from arising in _gyrA_. However, whether the _qnr_-genes were present before or after the mutation in _gyrA_ arose is unknown and need further investigation.

PMQR was detected in only 21% of the QREC isolates in article II and none of the isolates in article III. This is in concordance with other studies in Europe [@Veldman2011;@Borjesson2015;@deJong2018;@Wasyl2018]. _qnr_-genes have regularly been found on plasmids bearing additional resistance genes, such as extended spectrum $\beta$-lactamases (ESBL) [@Martinez-Martinez1998; @Wang2003; @Robicsek2006a; @Strahilevitz2009; @Jacoby2006a; @Zurfluh2015d]. PMQR could be an explanation for the presence of quinolone resistance without selective pressure, as these genes may be horizontally transmitted between bacteria, unlike chromosomal mutations. Additionally, PMQR usually confer low quinolone resistance on its own compared to chromosomal mutations. Therefore, PMQR could be a plausible explanation of the presence of QREC in Norwegian livestock, as most of the MIC values of the isolates were within 0.12 to 1 mg/L ciprofloxacin. However, the isolates in the present study had a low occurrence of PMQR genes, as well as ESBL and other acquired resistance genes. Therefore, this may indicate that quinolone resistance gene-bearing plasmids are rare in these animal populations in Norway, and only play a small role in the occurence of quinolone resistance in these animal environments.



### Possible associations between isolates from various origins

In Article II and III, the association between the included isolates were investigated by phylogenetic analysis. The aim this analysis was to identify possible dissemination of QREC strains, and to detect possible development of quinolone resistance in wild type _E. coli_ in the broiler production chain. In short, the results suggests dissemination of major STs through the broiler production chain, and little development of quinolone resistance among wild type _E. coli_. Furthermore, phylogenetically related isolates were detected among broilers and red foxes, from geograhically distant locations and from different years.

The major STs identified among QREC isolates from broilers were ST10, ST349 and ST355. These STs have previously been identified as major STs of QREC isolates in samples from broilers in Norway, Sweden and Iceland [@Myrenaas2018]. The broiler production environment is a highly stable environment with little temperature changes, closed to the outside, and is populated by a single animal species. It is therefore plausible to think that isolates within this environment have little reason to change, due to its stable nature. These major STs may therefore represent isolates that are highly adapted to the broiler production environment, and is likely to persist if introduced through imported animals. Parent animals are imported to Norway from Sweden, which is a country much like Norway in regards to quinolone usage. The overall sale of quinolones for animal use in Sweden have dropped 82% since 2009, and only 29 kg of active substance was sold in 2018 [@SWEDRES2019]. Additionally, a Swedish study have implicated imported breeding birds as the plausible source of QREC in their broiler production chain [@Borjesson2015]. This indicates that QREC isolates may originate from the top of the pyramid. Grandparent animals are imported from Scotland to Sweden, but no data on QREC occurrence or quinolone usage is available from this section of the production pyramid. Furthermore, phylogenetically related QREC from broilers and red foxes isolated in different years and geographic locations were identified, which may indicate persistence of some QREC strains in the broiler production. 

Phylogenetically related isolates from pigs were identified in Article II. These were isolates from geographically distant location, but within the same year. Since import of live animals to the pig production is negligible, the origin of these QREC isolates is likely within the Norwegian pig production. The presence of these isolates may indicate dissemination within the pig production chain as well, likely through either a shared piglet distributor or through satelite units of the sow pool system. 


Feed may be source of QREC [@Thorsteinsdottir2010]. A high diversity may indicate introduction of multiple strains into the production chain [@Bortolaia2010a].


## Methodological considerations

### Isolate selection
All isolates included in this study were collected through the NORM-VET programme, isolated with either the non-selective or selective method (see Section \@ref(norwmonitor)). The NORM-VET sampling regime ensures a representative selection of the actual _E. coli_ population in the broiler- and pig production chains, as well as wild animals, due to pre-defined sampling schemes. In the epidemiological study, all isolates of _E. coli_ and QREC from the NORM-VET programme between and including 2006 to 2016 was selected. Therefore, the included samples were deemed representative for the actual _E. coli_ populations in their respective animal species. In the QREC HTS study (article 2), selection was done to ensure phenotypic diversity of the isolates. Isolates were grouped based on phenotypic traits (MIC-values). Therefore, the observed diversity may not represent the actual genomic diversity in the QREC populations within each of the included animal species. In the broiler study, isolates were included if the location of origin was sampled at least three times, and that the location was represented with at least one QREC and one sensitive isolate.


### Genomic analyses methods
The various software used in the genomic analysis was selected because they are known and reliable, and most of them are implemented in the in-house pipeline Bifrost (github.com/NorwegianVeterinaryInstitute/Bifrost).

#### Contaminants
MALDI-TOF was used to confirm the species as _E. coli_ during DNA extraction. However, some samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. Inoculating impure samples may have resulted in extraction of DNA from multiple species, as some bacteria may have similar morphology on MacConkey media. This contamination resulted in the exclusion of four isolates due to high amounts of reads originating from these species. Reads from other bacterial species than _E. coli_ may also have been present in other isolates. However, filtering contigs smaller than 500 bp after assembly should remove most of the contaminant contigs, and increased the overall quality of the assemblies.

#### Identification of genes and mutations
In this project, ARIBA [@Hunt2017] was used to identify resistance genes, mutations in intrinsic genes related to resistance, and for sequence typing. For this work, ARIBA has multiple advantages, such as being able to detect both resistance genes and mutations. ARIBA is a read-based program, and using a read-based approach for the detection of AMR genes has been found to be superior to using assembled sequences [@Clausen2016]. The process of genome assembly is complicated, and if one gene is for example split over multiple contigs they may be missed [@Hunt2017;@Clausen2016]. Another advantage of using ARIBA is the possibility of downloading newly updated databases of our choosing. Here, the ResFinder [@Zankari2012] database was used for identification of acquired genes, and the MegaRes [@Lakin2017] database for mutations in intrinsic genes. Resfinder is a popular database for acquired resistance genes, which is manually curated and updated regularly. MEGARes was originally created for metagenomic characterization of AMR genes and mutations [@Lakin2017], and contains references from ResFinder, ARG-ANNOT, the Comprehensive Antibiotic Resistance Database (CARD), in addition to the National Center for Biotechnology Information (NCBI) Lahey clinic $\beta$-lactamase archive. MEGARes is also manually curated, and is specifically annotated for high-throughput data processing [@Lakin2017]. MEGARes was chosen as a database because it already encompasses other popular resistance databases, and is easily downloaded through ARIBA.

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel acquired genes is therefore missed. However, some mutations or genes may have been misclassified as not present due to the handling of the flags reported by ARIBA. ARIBA provides bitwise flags to assess the quality of the identified gene or mutation. In the present study, flags that did not represent the inclusion criteria were excluded from the final results. Thus, some isolates may have been categorized as negative for the given variant or gene, when they in reality may have not been.

Mutations in mutator genes were identified by comparing to the _E. coli_ K12 version of the respective gene. Comparing all isolates with only one _E. coli_ may not be appropriate, as different lineages of _E. coli_ may have different baseline versions of the genes. Therefore, differences between an isolate's sequence and the K12 sequence may have been categorized as mutations, when in reality the isolate's lineage may have this variant naturally. This uncertainty is not controlled for in the present study. However, comparing sequences to K12 may still give an indication of the variants that may naturally exist in the _E. coli_ population, indicating that specific lineages may have a naturally higher mutation rate. 

### Phylogeny
Phylogenetic methods were used to identify the evolutionary relationship between the isolates. An evolutionary approach was used because the isolates had no clear epidemiological link. In an outbreak situation, it is to be assumed that the isolates involved have a recent common ancestor - this may not be the case in situations where the isolates are not expected to be closely related. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and typing-based methods like MLST or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as typing-based methods do not take into account the amount of difference between alleles [@Feil2010]. Therefore, a phylogenetic approach was used to get a deeper resolution when determining the relationship between isolates.

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, which may have influenced the phylogenetic analysis. Pilon was used to correct errors by mapping the reads back to the assembly. However, even with this correction step, errors may persist and influence the topology or branch lengths in the final trees.

Removing recombinant sites is an important step when analysing vertical inheritance. Here, Gubbins was used in this regard, which have been found to be much faster than similar software [@Croucher2015], such as ClonalFrameML [@Didelot2015].

IQTree was selected as the ML algorithm to find the optimal phylogenetic tree. Comparative analyses have shown that the search strategy implemented in IQTree achieves higher likelihoods than similar algorithms [@Nguyen2015], such as RAxML [@Stamatakis2014] and PhyML [@Guindon2010], and IQTree seem to find optimal trees faster. Additionally, IQTree is easier to use for non-evolutionary biologists compared to RAxML, which was the main incentive in selecting IQTree. However, IQTree was not always the best algorithm in the comparative analysis between RAxML, PhyML and IQTree, and the authors recommended using all three programs [@Nguyen2015]. In this study, only IQTree was used, which means that potential trees with higher likelihoods may have been missed. 

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree for each phylogenetic analysis. ModelFinder finds the optimal evolutionary model for the data, based on criteria that control for overparameterization [@Kalyaanamoorthy2017]. This way, the data itself choose which model is appropriate, rather than the user. However, using more than one model may be necessary when working with many isolates from potentially different lineages, as some lineages may evolve differently than others. The selected models used in this study may therefore not be optimal in describing the evolutionary rate of all the isolates in one analysis.


### Statistical considerations

A non-parametric iteration test was used to assess the aggregation of samples in phylogenetic trees based on SNP-distances. A non-parametric test was selected because the underlying distribution of SNP-distances did not follow a theoretical distribution, such as a normal distribution. Therefore, a test that does not rely on distribution assumptions was needed. 


Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of resistance genes and mutations (Article II). NMDS is an ordination technique that differs from other ordination methods, such as principal component analysis (PCA), as NMDS does not assume a relationship between the samples [@Holland2008]. Therefore, any distance measure is allowed in NMDS, which makes this method well suited for a wide variety of data. NMDS is an iterative method that finds a solution after _n_ number of iterations or after an acceptable solution has been found. This means that running a separate NMDS analysis on the same data may yield a slightly different result. This is done to reduce the computational power needed in the analysis. However, this also means that the "best" solution may not be found, as the analysis may be stuck in local minima and only identifies the best solution of all nearby solutions [@Holland2008]. Therefore, the solution found in Article II may thus represent an adequate solution rather than the "best" one. A stress value is calculated for each iteration and represents how good the data is summarized by the ordination. In the present study, this stress value was low (< 0.05), which is regarded as a good representation [@Buttigieg2014a;@Buttigieg2014]. When running an NMDS analysis, it is important to select the amount of axes used in the calculation, as this may affect the results. In the present study, two axes were used (the default value), as no decrease in stress was observed by including more axes in the analysis. 
