# Discussion

## Methodological considerations

### Isolate selection
All isolates included in this study were collected through the NORM-VET programme, isolated with either the non-selective or selective method (see Section \@ref(norwmonitor)). The NORM-VET sampling regime ensures a representative selection of the actual _E. coli_ population in the broiler- and pig production chains, as well as wild animals, due to pre-defined sampling schemes. In the epidemiological study, all isolates of _E. coli_ and QREC from the NORM-VET programme between and including 2006 to 2016 was selected. Therefore, the included samples were deemed representative for the actual _E. coli_ populations in their respective animal species. In the QREC HTS study (article 2), selection was done to ensure phenotypic diversity of the isolates. Isolates were grouped based on phenotypic traits (MIC-values). Therefore, the observed diversity may not represent the actual genomic diversity in the QREC populations within each of the included animal species. In the broiler study, isolates were included if the location of origin was sampled at least three times, and that the location was represented with at least one QREC and one sensitive isolate.


### Genomic analyses methods
The various software used in the genomic analysis was selected because they are known and reliable, and most of them are implemented in the in-house pipeline Bifrost (https://github.com/NorwegianVeterinaryInstitute/Bifrost).

#### Contaminants
MALDI-TOF was used to confirm the species of the colonies as _E. coli_ during DNA extraction. However, some samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. Inoculating impure samples may have resulted in extraction of DNA from multiple species, as some bacteria may have similar morphology on MacConkey media. This contamination resulted in the exclusion of four isolates due to high amounts of reads originating from these species. Reads from other bacterial species than _E. coli_ may also have been present in other isolates. However, filtering contigs smaller than 500 bp after assembly should remove most of the contaminant contigs, and increased the overall quality of the assemblies.

#### Identification of genes and mutations
Both alignment-based methods (mapping) and assembly-based methods exist for the identification of resistance genes and variants. However, using alignment-based approaches for the detection of AMR genes have been shown to be superior to using assembled sequences [@Clausen2016], as the complicated process of _de novo_ genome assembly may lead to the AMR determinants being missed [@Hunt2017].
In this thesis, the resistance genes and variants were identified directly from the sequencing reads by using the Antimicrobial Resistance Identification By Assembly (ARIBA) tool [@Hunt2017]. Resistance towards quinolones in _E. coli_ is usually related to mutations in intrinsic genes, i.e. genes that are naturally present in _E. coli_ and doesn't confer resistance in its wild-type form. In addition, some PMQR genes are also related to quinolone resistance. Therefore, we needed a tool that could identify both mutations in intrinsic genes and plasmid-mediated genes at high accuracy and with the possibility of specifying which databases to use. ARIBA is a tool that fulfills these criteria.

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel acquired genes is therefore missed. In the case of mutations in intrinsic genes, all genes of interest in this study was included in the database used. However, some mutations may have been misclassified as not present due to the handling of the flags reported by ARIBA. ARIBA provides bitwise flags to assess the quality of the identified gene or mutation. In the present study, flags that did not represent the inclusion criteria were excluded from the final results. Thus, some isolates may have been categorized as negative for the given variant or gene, when they in reality may have not been. However, ensuring that the respective variant or gene was adequately assembled is important. For example, some isolates had the same mutation identified in two different references. This may be due to slight variations in the population from which the DNA was extracted from, resulting in the identification of two different closest references. These variants were filtered from the results due to a non-acceptable flag based on the lack of sequence coverage.

Mutations in mutator genes were identified by comparing to the _E. coli_ K12 version of the respective gene. Comparing all isolates with only one _E. coli_ may not be appropriate, as different lineages of _E. coli_ may have different baseline versions of the genes. Therefore, differences between an isolate's sequence and the K12 sequence may have been categorized as mutations, when in reality the isolate's lineage may have this variant naturally. This uncertainty is not controlled for in the present study. However, comparing sequences to K12 may still give an indication of the variants that may naturally exist in the _E. coli_ population, indicating that specific lineages may have a naturally higher mutation rate. 

### Phylogeny
Phylogenetic methods were used to identify the evolutionary relationship between the isolates. An evolutionary approach was used because the isolates had no clear epidemiological link. In an outbreak situation, it is to be assumed that the isolates involved have a recent common ancestor - this may not be the case in situations where the isolates are not expected to be closely related. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and typing-based methods like MLST or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as typing-based methods do not take into account the amount of difference between alleles [@Feil2010].Therefore, a phylogenetic approach was used to get a deeper resolution when determining the relationship between isolates.

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, which may have influenced the phylogenetic analysis. Pilon was used to correct errors by mapping the reads back to the assembly. However, even with this correction step, errors may persist and influence the topology or branch lengths in the final trees. 

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree [@Nguyen2015] for each phylogenetic analysis. ModelFinder finds the optimal evolutionary model for the data, based on criteria that control for overparameterization [@Kalyaanamoorthy2017]. This way, the data itself choose which model is appropriate, rather than the user. However, using more than one model may be necessary when working with many isolates from potentially different lineages, as some lineages may evolve differently than others. The selected models used in this study may therefore not be optimal in describing the evolutionary rate of all the isolates at once.



## Main results discussion

### Identification of occurrence of quinolone resistance in various production compartments and the environment
The occurrence of quinolone resistance, i.e. QREC, was identified by compiling and analysing NORM-VET data from 2006 to 2016, and relating this to human population density. Here, QREC occurrence was determined in samples from multiple species, including broilers, layers, pigs, turkeys, horses, cattle, sheep, dogs, reindeer, red foxes, and different species of wild birds.




### Characterization of genomic factors

### Identification of possible associations between isolates from various origins

### Investigate possible risk factors




