# Materials and Methods

Below follows a brief summary of methods used in this study. For detailed methodology, see the attached articles.

## In-vitro methodology

### Screening for QREC

- Selective/Non-selective methodology
- Sampling
- MIC-determination (SensiTitre)
- PCR and Sanger Sequencing

### Sequencing

- DNA extraction
- Quality control
- Sequencing

## In-silico methodology

- Bifrost pipeline:
    + Quality control
    + Antimicrobial resistance gene detection and MLST
    + Assembly and annotation
  
- Phylogeny
    + Core genome MLST
    + Alignment with mauve
    + Gubbins
    + IQTree

## Statistics

## R scripts
Various R-scripts have been used to analyse multiple datasets in this project. These are available on github at www.github.com/hkaspersen, and were written with `r version$version.string`.